diff --git a/src/NS/SentinelBundle/Entity/BaseCase.php b/src/NS/SentinelBundle/Entity/BaseCase.php
index b5c87c8..c359fc9 100644
--- a/src/NS/SentinelBundle/Entity/BaseCase.php
+++ b/src/NS/SentinelBundle/Entity/BaseCase.php
@@ -455,7 +455,7 @@ abstract class BaseCase
      */
     public function getSentToReferenceLab()
     {
-        return ($this->siteLab) ? $this->siteLab->getSentToReferenceLab() : false;
+        return (($this->siteLab && $this->siteLab->getSentToReferenceLab()) || ($this->nationalLab && $this->nationalLab->getSentToReferenceLab()));
     }
 
     /**
diff --git a/src/NS/SentinelBundle/Entity/BaseExternalLab.php b/src/NS/SentinelBundle/Entity/BaseExternalLab.php
index 36db8ac..20efbb2 100644
--- a/src/NS/SentinelBundle/Entity/BaseExternalLab.php
+++ b/src/NS/SentinelBundle/Entity/BaseExternalLab.php
@@ -15,7 +15,7 @@ use JMS\Serializer\Annotation as Serializer;
 abstract class BaseExternalLab
 {
     /**
-     * @var
+     * @var BaseCase
      */
     protected $caseFile;
 
@@ -64,9 +64,6 @@ abstract class BaseExternalLab
      */
     protected $comment;
 
-    /**
-     *
-     */
     public function __construct()
     {
         $this->status = new CaseStatus(0);
@@ -74,16 +71,6 @@ abstract class BaseExternalLab
     }
 
     /**
-     * Get id
-     *
-     * @return integer
-     */
-    public function getId()
-    {
-        return $this->id;
-    }
-
-    /**
      * @return \DateTime
      */
     public function getDateReceived()
diff --git a/src/NS/SentinelBundle/Entity/IBD.php b/src/NS/SentinelBundle/Entity/IBD.php
index c259bdb..cef3bc0 100644
--- a/src/NS/SentinelBundle/Entity/IBD.php
+++ b/src/NS/SentinelBundle/Entity/IBD.php
@@ -385,6 +385,7 @@ class IBD extends BaseCase
     private $blood_collected;
 
     /**
+     * TODO Multiply by 2 for 2 samples
      * @var \DateTime $bloodCollectDate
      * @ORM\Column(name="blood_collect_date",type="date",nullable=true)
      * @Serializer\Groups({"api","export"})
diff --git a/src/NS/SentinelBundle/Entity/IBD/ExternalLab.php b/src/NS/SentinelBundle/Entity/IBD/ExternalLab.php
index 7afaa18..4f70ec6 100644
--- a/src/NS/SentinelBundle/Entity/IBD/ExternalLab.php
+++ b/src/NS/SentinelBundle/Entity/IBD/ExternalLab.php
@@ -717,7 +717,7 @@ abstract class ExternalLab extends BaseExternalLab
     }
 
     /**
-     * @return null|string|void
+     * @return null|string
      */
     public function getIncompleteField()
     {
diff --git a/src/NS/SentinelBundle/Entity/IBD/NationalLab.php b/src/NS/SentinelBundle/Entity/IBD/NationalLab.php
index 81ea217..3045d4e 100644
--- a/src/NS/SentinelBundle/Entity/IBD/NationalLab.php
+++ b/src/NS/SentinelBundle/Entity/IBD/NationalLab.php
@@ -3,6 +3,8 @@
 namespace NS\SentinelBundle\Entity\IBD;
 
 use Doctrine\ORM\Mapping as ORM;
+use JMS\Serializer\Annotation as Serializer;
+use NS\SentinelBundle\Validators as LocalAssert;
 
 /**
  * Description of ReferenceLab
@@ -22,6 +24,63 @@ class NationalLab extends ExternalLab
     protected $caseFile;
 
     /**
+     * @var boolean
+     * @ORM\Column(name="rl_isol_csf_sent",type="boolean",nullable=true)
+     *
+     * @LocalAssert\NoFutureDate()
+     * @Serializer\Groups({"api","export"})
+     */
+    private $rl_isol_csf_sent;
+
+    /**
+     * @var \DateTime
+     * @ORM\Column(name="rl_isol_csf_date",type="date",nullable=true)
+     *
+     * @LocalAssert\NoFutureDate()
+     * @Serializer\Groups({"api","export"})
+     * @Serializer\Type(name="DateTime<'Y-m-d'>")
+     */
+    private $rl_isol_csf_date;
+
+    /**
+     * @var boolean
+     * @ORM\Column(name="rl_isol_blood_sent",type="boolean",nullable=true)
+     *
+     * @LocalAssert\NoFutureDate()
+     * @Serializer\Groups({"api","export"})
+     */
+    private $rl_isol_blood_sent;
+
+    /**
+     * @var \DateTime
+     * @ORM\Column(name="rl_isol_blood_date",type="date",nullable=true)
+     *
+     * @LocalAssert\NoFutureDate()
+     * @Serializer\Groups({"api","export"})
+     * @Serializer\Type(name="DateTime<'Y-m-d'>")
+     */
+    private $rl_isol_blood_date;
+
+    /**
+     * @var boolean
+     * @ORM\Column(name="rl_other_sent",type="boolean",nullable=true)
+     *
+     * @LocalAssert\NoFutureDate()
+     * @Serializer\Groups({"api","export"})
+     */
+    private $rl_other_sent;
+
+    /**
+     * @var \DateTime
+     * @ORM\Column(name="rl_other_date",type="date",nullable=true)
+     *
+     * @LocalAssert\NoFutureDate()
+     * @Serializer\Groups({"api","export"})
+     * @Serializer\Type(name="DateTime<'Y-m-d'>")
+     */
+    private $rl_other_date;
+
+    /**
      * @var string
      */
     private $type        = 'NL';
@@ -61,4 +120,108 @@ class NationalLab extends ExternalLab
         $this->caseFile = $caseFile;
         return $this;
     }
+
+    /**
+     * @return bool
+     */
+    public function isRlIsolCsfSent()
+    {
+        return $this->rl_isol_csf_sent;
+    }
+
+    /**
+     * @param bool $rl_isol_csf_sent
+     */
+    public function setRlIsolCsfSent($rl_isol_csf_sent)
+    {
+        $this->rl_isol_csf_sent = $rl_isol_csf_sent;
+    }
+
+    /**
+     * @return \DateTime
+     */
+    public function getRlIsolCsfDate()
+    {
+        return $this->rl_isol_csf_date;
+    }
+
+    /**
+     * @param \DateTime $rl_isol_csf_date
+     */
+    public function setRlIsolCsfDate(\DateTime $rl_isol_csf_date = null)
+    {
+        $this->rl_isol_csf_date = $rl_isol_csf_date;
+    }
+
+    /**
+     * @return bool
+     */
+    public function isRlIsolBloodSent()
+    {
+        return $this->rl_isol_blood_sent;
+    }
+
+    /**
+     * @param bool $rl_isol_blood_sent
+     */
+    public function setRlIsolBloodSent($rl_isol_blood_sent)
+    {
+        $this->rl_isol_blood_sent = $rl_isol_blood_sent;
+    }
+
+    /**
+     * @return \DateTime
+     */
+    public function getRlIsolBloodDate()
+    {
+        return $this->rl_isol_blood_date;
+    }
+
+    /**
+     * @param \DateTime $rl_isol_blood_date
+     */
+    public function setRlIsolBloodDate(\DateTime $rl_isol_blood_date = null)
+    {
+        $this->rl_isol_blood_date = $rl_isol_blood_date;
+    }
+
+    /**
+     * @return bool
+     */
+    public function isRlOtherSent()
+    {
+        return $this->rl_other_sent;
+    }
+
+    /**
+     * @param bool $rl_other_sent
+     */
+    public function setRlOtherSent($rl_other_sent)
+    {
+        $this->rl_other_sent = $rl_other_sent;
+    }
+
+    /**
+     * @return \DateTime
+     */
+    public function getRlOtherDate()
+    {
+        return $this->rl_other_date;
+    }
+
+    /**
+     * @param \DateTime $rl_other_date
+     */
+    public function setRlOtherDate(\DateTime $rl_other_date = null)
+    {
+        $this->rl_other_date = $rl_other_date;
+    }
+
+    /**
+     * @return bool
+     */
+    public function getSentToReferenceLab()
+    {
+        return ($this->rl_isol_csf_sent || $this->rl_isol_blood_sent || $this->rl_other_sent);
+    }
 }
diff --git a/src/NS/SentinelBundle/Entity/IBD/SiteLab.php b/src/NS/SentinelBundle/Entity/IBD/SiteLab.php
index 8d0c76b..d27cf91 100644
--- a/src/NS/SentinelBundle/Entity/IBD/SiteLab.php
+++ b/src/NS/SentinelBundle/Entity/IBD/SiteLab.php
@@ -1917,7 +1917,7 @@ class SiteLab implements BaseSiteLabInterface
     /**
      * @param TripleChoice $pleural_fluid_culture_done
      */
-    public function setPleuralFluidCultureDone($pleural_fluid_culture_done)
+    public function setPleuralFluidCultureDone(TripleChoice $pleural_fluid_culture_done)
     {
         $this->pleural_fluid_culture_done = $pleural_fluid_culture_done;
     }
@@ -1933,7 +1933,7 @@ class SiteLab implements BaseSiteLabInterface
     /**
      * @param CultureResult $pleural_fluid_culture_result
      */
-    public function setPleuralFluidCultureResult($pleural_fluid_culture_result)
+    public function setPleuralFluidCultureResult(CultureResult $pleural_fluid_culture_result)
     {
         $this->pleural_fluid_culture_result = $pleural_fluid_culture_result;
     }
@@ -1965,7 +1965,7 @@ class SiteLab implements BaseSiteLabInterface
     /**
      * @param TripleChoice $pleural_fluid_gram_done
      */
-    public function setPleuralFluidGramDone($pleural_fluid_gram_done)
+    public function setPleuralFluidGramDone(TripleChoice $pleural_fluid_gram_done)
     {
         $this->pleural_fluid_gram_done = $pleural_fluid_gram_done;
     }
@@ -1981,7 +1981,7 @@ class SiteLab implements BaseSiteLabInterface
     /**
      * @param GramStain $pleural_fluid_gram_result
      */
-    public function setPleuralFluidGramResult($pleural_fluid_gram_result)
+    public function setPleuralFluidGramResult(GramStain $pleural_fluid_gram_result)
     {
         $this->pleural_fluid_gram_result = $pleural_fluid_gram_result;
     }
@@ -1997,7 +1997,7 @@ class SiteLab implements BaseSiteLabInterface
     /**
      * @param GramStainResult $pleural_fluid_gram_result_organism
      */
-    public function setPleuralFluidGramResultOrganism($pleural_fluid_gram_result_organism)
+    public function setPleuralFluidGramResultOrganism(GramStainResult $pleural_fluid_gram_result_organism)
     {
         $this->pleural_fluid_gram_result_organism = $pleural_fluid_gram_result_organism;
     }
@@ -2013,7 +2013,7 @@ class SiteLab implements BaseSiteLabInterface
     /**
      * @param TripleChoice $pleural_fluid_pcr_done
      */
-    public function setPleuralFluidPcrDone($pleural_fluid_pcr_done)
+    public function setPleuralFluidPcrDone(TripleChoice $pleural_fluid_pcr_done)
     {
         $this->pleural_fluid_pcr_done = $pleural_fluid_pcr_done;
     }
@@ -2029,7 +2029,7 @@ class SiteLab implements BaseSiteLabInterface
     /**
      * @param PCRResult $pleural_fluid_pcr_result
      */
-    public function setPleuralFluidPcrResult($pleural_fluid_pcr_result)
+    public function setPleuralFluidPcrResult(PCRResult $pleural_fluid_pcr_result)
     {
         $this->pleural_fluid_pcr_result = $pleural_fluid_pcr_result;
     }
@@ -2061,7 +2061,7 @@ class SiteLab implements BaseSiteLabInterface
     /**
      * @param \DateTime $blood_lab_date
      */
-    public function setBloodLabDate($blood_lab_date)
+    public function setBloodLabDate(\DateTime $blood_lab_date = null)
     {
         $this->blood_lab_date = $blood_lab_date;
     }
@@ -2077,7 +2077,7 @@ class SiteLab implements BaseSiteLabInterface
     /**
      * @param \DateTime $blood_lab_time
      */
-    public function setBloodLabTime($blood_lab_time)
+    public function setBloodLabTime(\DateTime $blood_lab_time = null)
     {
         $this->blood_lab_time = $blood_lab_time;
     }
@@ -2109,7 +2109,7 @@ class SiteLab implements BaseSiteLabInterface
     /**
      * @param \DateTime $other_lab_date
      */
-    public function setOtherLabDate($other_lab_date)
+    public function setOtherLabDate(\DateTime $other_lab_date = null)
     {
         $this->other_lab_date = $other_lab_date;
     }
@@ -2125,7 +2125,7 @@ class SiteLab implements BaseSiteLabInterface
     /**
      * @param \DateTime $other_lab_time
      */
-    public function setOtherLabTime($other_lab_time)
+    public function setOtherLabTime(\DateTime $other_lab_time = null)
     {
         $this->other_lab_time = $other_lab_time;
     }
diff --git a/src/NS/SentinelBundle/Form/IBD/NationalLabType.php b/src/NS/SentinelBundle/Form/IBD/NationalLabType.php
index acf02b0..6c3a3a1 100644
--- a/src/NS/SentinelBundle/Form/IBD/NationalLabType.php
+++ b/src/NS/SentinelBundle/Form/IBD/NationalLabType.php
@@ -2,12 +2,67 @@
 
 namespace NS\SentinelBundle\Form\IBD;
 
+use NS\AceBundle\Form\SwitchType;
+use NS\AceBundle\Form\DatePickerType;
+use NS\SentinelBundle\Entity\Country;
 use NS\SentinelBundle\Entity\IBD\NationalLab;
+use NS\SentinelBundle\Entity\Site;
+use NS\SentinelBundle\Services\SerializedSites;
 use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormEvent;
+use Symfony\Component\Form\FormEvents;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
 class NationalLabType extends AbstractType
 {
+    /** @var SerializedSites */
+    private $siteSerializer;
+
+    /**
+     * NationalLabType constructor.
+     * @param SerializedSites $siteSerializer
+     */
+    public function __construct(SerializedSites $siteSerializer)
+    {
+        $this->siteSerializer = $siteSerializer;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder->addEventListener(FormEvents::POST_SET_DATA,[$this,'postSetData']);
+    }
+
+    public function postSetData(FormEvent $event)
+    {
+        $data = $event->getData();
+        $form = $event->getForm();
+        $country = null;
+
+        if ($data && $data->getCaseFile() && $data->getCaseFile()->getCountry()) {
+            $country = $data->getCaseFile()->getCountry();
+        } elseif (!$this->siteSerializer->hasMultipleSites()) {
+            $site = $this->siteSerializer->getSite();
+            $country = ($site instanceof Site) ? $site->getCountry() : null;
+        }
+
+        if ($country instanceof Country) {
+            if ($country->hasReferenceLab()) {
+                $form
+                    ->add('rlIsolCsfSent', SwitchType::class, ['label' => 'ibd-form.csf-isol-sent-to-rrl', 'required' => false])
+                    ->add('rlIsolCsfDate', DatePickerType::class, ['label' => 'ibd-form.csf-isol-sent-to-rrl-date', 'required' => false, 'hidden' => ['parent' => 'rlIsolCsfSent', 'value' => 1]])
+                    ->add('rlIsolBloodSent', SwitchType::class, ['label' => 'ibd-form.blood-sent-to-rrl', 'required' => false])
+                    ->add('rlIsolBloodDate', DatePickerType::class, ['label' => 'ibd-form.blood-sent-to-rrl-date', 'required' => false, 'hidden' => ['parent' => 'rlIsolBloodSent', 'value' => 1]])
+                    ->add('rlOtherSent', SwitchType::class, ['label' => 'ibd-form.other-sent-to-rrl', 'required' => false])
+                    ->add('rlOtherDate', DatePickerType::class, ['label' => 'ibd-form.other-sent-to-rrl-date', 'required' => false, 'hidden' => ['parent' => 'rlOtherSent', 'value' => 1]]);
+            }
+
+        }
+    }
+
     /**
      * @param OptionsResolver $resolver
      */
diff --git a/src/NS/SentinelBundle/Form/IBD/SiteLabType.php b/src/NS/SentinelBundle/Form/IBD/SiteLabType.php
index 4d821cd..bb32d17 100644
--- a/src/NS/SentinelBundle/Form/IBD/SiteLabType.php
+++ b/src/NS/SentinelBundle/Form/IBD/SiteLabType.php
@@ -145,20 +145,6 @@ class SiteLabType extends AbstractType
         }
 
         if ($country instanceof Country) {
-            if ($country->hasReferenceLab()) {
-                $form
-                    ->add('rlCsfSent', SwitchType::class, ['label' => 'ibd-form.csf-sent-to-rrl', 'required' => false, 'switch_type' => 2])
-                    ->add('rlCsfDate', DatePickerType::class, ['label' => 'ibd-form.csf-sent-to-rrl-date', 'required' => false, 'hidden' => ['parent' => 'rlCsfSent', 'value' => 1]])
-                    ->add('rlIsolCsfSent', SwitchType::class, ['label' => 'ibd-form.csf-isol-sent-to-rrl', 'required' => false])
-                    ->add('rlIsolCsfDate', DatePickerType::class, ['label' => 'ibd-form.csf-isol-sent-to-rrl-date', 'required' => false, 'hidden' => ['parent' => 'rlIsolCsfSent', 'value' => 1]])
-                    ->add('rlIsolBloodSent', SwitchType::class, ['label' => 'ibd-form.blood-sent-to-rrl', 'required' => false])
-                    ->add('rlIsolBloodDate', DatePickerType::class, ['label' => 'ibd-form.blood-sent-to-rrl-date', 'required' => false, 'hidden' => ['parent' => 'rlIsolBloodSent', 'value' => 1]])
-                    ->add('rlBrothSent', SwitchType::class, ['label' => 'ibd-form.broth-sent-to-rrl', 'required' => false])
-                    ->add('rlBrothDate', DatePickerType::class, ['label' => 'ibd-form.broth-sent-to-rrl-date', 'required' => false, 'hidden' => ['parent' => 'rlBrothSent', 'value' => 1]])
-                    ->add('rlOtherSent', SwitchType::class, ['label' => 'ibd-form.other-sent-to-rrl', 'required' => false])
-                    ->add('rlOtherDate', DatePickerType::class, ['label' => 'ibd-form.other-sent-to-rrl-date', 'required' => false, 'hidden' => ['parent' => 'rlOtherSent', 'value' => 1]]);
-            }
-
             if ($country->hasNationalLab()) {
                 $form
                     ->add('nlCsfSent', SwitchType::class, ['label' => 'ibd-form.csf-sent-to-nl', 'required' => false, 'switch_type' => 2])
@@ -167,8 +153,6 @@ class SiteLabType extends AbstractType
                     ->add('nlIsolCsfDate', DatePickerType::class, ['label' => 'ibd-form.csf-isol-sent-to-nl-date', 'required' => false, 'hidden' => ['parent' => 'nlIsolCsfSent', 'value' => 1]])
                     ->add('nlIsolBloodSent', SwitchType::class, ['label' => 'ibd-form.blood-sent-to-nl', 'required' => false])
                     ->add('nlIsolBloodDate', DatePickerType::class, ['label' => 'ibd-form.blood-sent-to-nl-date', 'required' => false, 'hidden' => ['parent' => 'nlIsolBloodSent', 'value' => 1]])
-                    ->add('nlBrothSent', SwitchType::class, ['label' => 'ibd-form.broth-sent-to-nl', 'required' => false])
-                    ->add('nlBrothDate', DatePickerType::class, ['label' => 'ibd-form.broth-sent-to-nl-date', 'required' => false, 'hidden' => ['parent' => 'nlBrothSent', 'value' => 1]])
                     ->add('nlOtherSent', SwitchType::class, ['label' => 'ibd-form.other-sent-to-nl', 'required' => false])
                     ->add('nlOtherDate', DatePickerType::class, ['label' => 'ibd-form.other-sent-to-nl-date', 'required' => false, 'hidden' => ['parent' => 'nlOtherSent', 'value' => 1]]);
             }
diff --git a/src/NS/SentinelBundle/Resources/config/forms.yml b/src/NS/SentinelBundle/Resources/config/forms.yml
index dd19fa9..191e28e 100644
--- a/src/NS/SentinelBundle/Resources/config/forms.yml
+++ b/src/NS/SentinelBundle/Resources/config/forms.yml
@@ -160,6 +160,7 @@ services:
 
   ns.sentinel.form.type.ibd_nl:
     class: NS\SentinelBundle\Form\IBD\NationalLabType
+    arguments: [ "@ns.sentinel.sites" ]
     tags:
       - { name: form.type }
 
diff --git a/src/NS/SentinelBundle/Resources/views/IBD/editLab.html.twig b/src/NS/SentinelBundle/Resources/views/IBD/editLab.html.twig
index 19042f2..5702145 100644
--- a/src/NS/SentinelBundle/Resources/views/IBD/editLab.html.twig
+++ b/src/NS/SentinelBundle/Resources/views/IBD/editLab.html.twig
@@ -221,8 +221,6 @@
                             <div class="row">
                                 <div class="col-sm-3">{{ form_row(form.nlIsolBloodSent) }}</div>
                                 <div class="col-sm-3">{{ form_row(form.nlIsolBloodDate) }}</div>
-                                <div class="col-sm-3">{{ form_row(form.nlBrothSent) }}</div>
-                                <div class="col-sm-3">{{ form_row(form.nlBrothDate) }}</div>
                             </div>
                         {% endif %}
                     </div>
@@ -312,13 +310,13 @@
                             </div>
                         </div>
                         <hr/>
-                        {% if form.rlOtherSent %}
+                        {% if form.rlOtherSent is defined %}
                             <div class="row">
                                 <div class="col-sm-3">{{ form_row(form.rlOtherSent) }}</div>
                                 <div class="col-sm-3">{{ form_row(form.rlOtherDate) }}</div>
                             </div>
                         {% endif %}
-                        {% if form.nlOtherSent %}
+                        {% if form.nlOtherSent is defined %}
                             <div class="row">
                                 <div class="col-sm-3">{{ form_row(form.nlOtherSent) }}</div>
                                 <div class="col-sm-3">{{ form_row(form.nlOtherDate) }}</div>
