diff --git a/src/NS/SentinelBundle/Resources/translations/messages.en.xliff b/src/NS/SentinelBundle/Resources/translations/messages.en.xliff
index 580dc66..40d8c97 100644
--- a/src/NS/SentinelBundle/Resources/translations/messages.en.xliff
+++ b/src/NS/SentinelBundle/Resources/translations/messages.en.xliff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:jms="urn:jms:translation" version="1.2">
-  <file date="2016-03-25T10:43:58Z" source-language="en" target-language="en" datatype="plaintext" original="not.available">
+  <file date="2016-03-28T16:56:03Z" source-language="en" target-language="en" datatype="plaintext" original="not.available">
     <header>
       <tool tool-id="JMSTranslationBundle" tool-name="JMSTranslationBundle" tool-version="1.1.0-DEV"/>
       <note>The source node in most cases contains the sample message as written by the developer. If it looks like a dot-delimitted string such as "form.label.firstname", then the developer has not provided a default message.</note>
@@ -1679,7 +1679,7 @@
       </trans-unit>
       <trans-unit id="bb9ba6fe87ecaac105f0f4af2c061a1748acd534" resname="external-lab-form.date-received">
         <source>external-lab-form.date-received</source>
-        <target>Date Sample received</target>
+        <target>Date sample received</target>
         <jms:reference-file line="25">Form/Rota/BaseLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="187902eabaf16101d65cb759b954624d7a98de8d" resname="external-lab-form.genotyping-date">
@@ -1950,7 +1950,7 @@
       </trans-unit>
       <trans-unit id="5bd6a0e287156c72e1ca281ce63053256252457c" resname="ibd-form.csf-lab-datetime">
         <source>ibd-form.csf-lab-datetime</source>
-        <target state="new">ibd-form.csf-lab-datetime</target>
+        <target>Date when CSF was received at the lab</target>
         <jms:reference-file line="48">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="09822becef86dd896f55f4132c0039831a11910e" resname="ibd-form.csf-lat-done">
@@ -2010,7 +2010,7 @@
       </trans-unit>
       <trans-unit id="1df7f169999de566518bcc5ce880d14ff589a65e" resname="ibd-form.csfDateTime">
         <source>ibd-form.csfDateTime</source>
-        <target state="new">ibd-form.csfDateTime</target>
+        <target>Date when CSF was received at the lab</target>
         <jms:reference-file line="216">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="c92accbe22abf53acac506dcede212ab0b061bca" resname="ibd-form.csfSiteId">
@@ -2107,7 +2107,7 @@
       </trans-unit>
       <trans-unit id="0da45bd56fa56862eebf9da5bebb9c33d58b4081" resname="ibd-form.hi-serotype-other">
         <source>ibd-form.hi-serotype-other</source>
-        <target state="new">ibd-form.hi-serotype-other</target>
+        <target>HiSerotype Other</target>
         <jms:reference-file line="109">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="49d1ec463de133d2d2ecb102a8f7f16395fbb0a3" resname="ibd-form.hib-doses">
@@ -2237,7 +2237,7 @@
       </trans-unit>
       <trans-unit id="6914b1791ddf8fcc5be2232c42dd5fa8a16a3e2d" resname="ibd-form.other-cult-done1">
         <source>ibd-form.other-cult-done1</source>
-        <target state="new">ibd-form.other-cult-done1</target>
+        <target>Were any other cultures done?</target>
         <jms:reference-file line="154">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="2237f5d2f7a05e8620bc7e1441b172f8b5d9887a" resname="ibd-form.other-cult-other">
@@ -2259,12 +2259,12 @@
       </trans-unit>
       <trans-unit id="025152f2d8874ff156c638b8d12c06e8675d774f" resname="ibd-form.other-test-done1">
         <source>ibd-form.other-test-done1</source>
-        <target state="new">ibd-form.other-test-done1</target>
+        <target>Were other test(s) performed on the specimen in the lab?</target>
         <jms:reference-file line="166">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="9fafeb6751ed03da89cf84d057e0afe72477b83e" resname="ibd-form.other-test-other">
         <source>ibd-form.other-test-other</source>
-        <target state="new">ibd-form.other-test-other</target>
+        <target>If other organism(s) were identified in the other test performed, please specify.</target>
         <jms:reference-file line="174">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="dac15dc0e75761c8518c20cfd8b4c291e4ae66b9" resname="ibd-form.other-test-result">
@@ -2468,7 +2468,7 @@
       </trans-unit>
       <trans-unit id="9995383f100098aee0b4cf066ec2462807012684" resname="ibd-rrl-form.rNasP">
         <source>ibd-rrl-form.rNasP</source>
-        <target>Results form rNaseP</target>
+        <target>Results of RNaseP (ct value)</target>
         <jms:reference-file line="53">Form/IBD/BaseLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="d6fe0780dce1527df3ca7938556bb512e466249c" resname="ibd-rrl-form.sample-collection-date">
@@ -2509,13 +2509,13 @@
       </trans-unit>
       <trans-unit id="61289dcc293d6f02ff71a837bb4581d20d1316f2" resname="ibd.bexA">
         <source>ibd.bexA</source>
-        <target>bexA</target>
+        <target>Results of bexA (ct value)</target>
         <jms:reference-file line="291">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="325">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="7b50728b4f1d1b8a67a7908e3baa16d62473af8e" resname="ibd.ctrA">
         <source>ibd.ctrA</source>
-        <target>ctrA</target>
+        <target>Results of ctrA (ct value)</target>
         <jms:reference-file line="287">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="321">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
@@ -2533,13 +2533,13 @@
       </trans-unit>
       <trans-unit id="0db7c9647874f109b059f6b651750b600d606026" resname="ibd.hpd1">
         <source>ibd.hpd1</source>
-        <target>hpd1</target>
+        <target>Results of hpd1 (ct value)</target>
         <jms:reference-file line="289">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="323">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="9b4df6fb6dd0179349ae6c86df0ca0bfa6f708f5" resname="ibd.hpd3">
         <source>ibd.hpd3</source>
-        <target>hpd3</target>
+        <target>Results of hpd3 (ct value)</target>
         <jms:reference-file line="290">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="324">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
@@ -2557,7 +2557,7 @@
       </trans-unit>
       <trans-unit id="d229fd5c48c36bce8d007a0dab2808643f24e316" resname="ibd.lytA">
         <source>ibd.lytA</source>
-        <target>lytA</target>
+        <target>Results of lytA (ct value)</target>
         <jms:reference-file line="286">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="320">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
@@ -2569,19 +2569,19 @@
       </trans-unit>
       <trans-unit id="ecdd9e5aef58c2090d94b2da337e4075230e92a5" resname="ibd.pathogenIdentifierMethod">
         <source>ibd.pathogenIdentifierMethod</source>
-        <target>ibd.pathogenIdentifierMethod</target>
+        <target>Method used for pathogen identification at RL</target>
         <jms:reference-file line="282">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="316">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="78c3ea1f4048bf6a83056df86cee076e667869c7" resname="ibd.pathogenIdentifierOther">
         <source>ibd.pathogenIdentifierOther</source>
-        <target>ibd.pathogenIdentifierOther</target>
+        <target>If other, please specify Method used for pathogen identification</target>
         <jms:reference-file line="283">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="317">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="b1ee9213cdfc0edaa8e803ff95618b0740b74e21" resname="ibd.rNaseP">
         <source>ibd.rNaseP</source>
-        <target>rNaseP</target>
+        <target>Results of RNaseP (ct value)</target>
         <jms:reference-file line="292">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="326">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
@@ -2605,7 +2605,7 @@
       </trans-unit>
       <trans-unit id="348d6dd2724fecb3c18ecdc8a685e75edf328c6f" resname="ibd.sodC">
         <source>ibd.sodC</source>
-        <target>sodC</target>
+        <target>Results of sodC (ct value)</target>
         <jms:reference-file line="288">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="322">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
