diff --git a/src/NS/SentinelBundle/Resources/translations/messages.es.xliff b/src/NS/SentinelBundle/Resources/translations/messages.es.xliff
index 357f148..09d65d5 100644
--- a/src/NS/SentinelBundle/Resources/translations/messages.es.xliff
+++ b/src/NS/SentinelBundle/Resources/translations/messages.es.xliff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:jms="urn:jms:translation" version="1.2">
-  <file date="2016-03-25T18:34:46Z" source-language="en" target-language="es" datatype="plaintext" original="not.available">
+  <file date="2016-03-28T15:52:44Z" source-language="en" target-language="es" datatype="plaintext" original="not.available">
     <header>
       <tool tool-id="JMSTranslationBundle" tool-name="JMSTranslationBundle" tool-version="1.1.0-DEV"/>
       <note>The source node in most cases contains the sample message as written by the developer. If it looks like a dot-delimitted string such as "form.label.firstname", then the developer has not provided a default message.</note>
@@ -13,7 +13,7 @@
       </trans-unit>
       <trans-unit id="18ab4daacc3850d1f77d27057bb1ec6d69b46aa7" resname="%count% Results">
         <source>%count% Results</source>
-        <target>Resultados: %count%</target>
+        <target>Resultados: %cantidad%</target>
         <jms:reference-file line="9">views/Report/fieldPopulation.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="c04cd8a70157ece1018ac829accacdeaeb9679c9" resname="0-5">
@@ -218,7 +218,7 @@
       </trans-unit>
       <trans-unit id="05fcdf14c20820118b7c12ffc13c5fdf639790bb" resname="ACW135 (polysaccharide)">
         <source>ACW135 (polysaccharide)</source>
-        <target>ACW135 (polysaccharide)</target>
+        <target>ACW135 (polisacárido)</target>
       </trans-unit>
       <trans-unit id="44708d6bef8db5753893523e34ea079d400e1e11" resname="ACYW135 (conjugate)">
         <source>ACYW135 (conjugate)</source>
@@ -226,7 +226,7 @@
       </trans-unit>
       <trans-unit id="61741390be197c1dd7ded938e4b990fb050a9d6b" resname="ACYW135 (polysaccharide)">
         <source>ACYW135 (polysaccharide)</source>
-        <target>ACYW135 (polysaccharide)</target>
+        <target>ACYW135 (polisacárido)</target>
       </trans-unit>
       <trans-unit id="c3cd636a585b20c40ac2df5ffb403e83cb2eef51" resname="Actions">
         <source>Actions</source>
@@ -249,11 +249,11 @@
       </trans-unit>
       <trans-unit id="5f0404ee8d2a4c6592f295f92613e391ecad291d" resname="Air bronchogram">
         <source>Air bronchogram</source>
-        <target>Broncograma aereo</target>
+        <target>Broncograma aéreo</target>
       </trans-unit>
       <trans-unit id="e6a9bb4fd705c50b883a4ccfb7fdc2c4e47f13a5" resname="Api Resources">
         <source>Api Resources</source>
-        <target>Api Resources</target>
+        <target>Recursos Api</target>
         <jms:reference-file line="52">SentinelBundle/Menu/Builder.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="ae4f281df5a5d0ff3cad6371f76d5c29b6d953ec" resname="B">
@@ -266,11 +266,11 @@
       </trans-unit>
       <trans-unit id="ffac615e11c50e4241d2a7eaf146969c132243dc" resname="Bacterial meningitis">
         <source>Bacterial meningitis</source>
-        <target>meningitis bacteriana</target>
+        <target>Meningitis bacteriana</target>
       </trans-unit>
       <trans-unit id="d3187331a49581538947126ee48c73b537af3b42" resname="Bacterial pneumonia">
         <source>Bacterial pneumonia</source>
-        <target>neumonía bacteriana</target>
+        <target>Neumonía bacteriana</target>
       </trans-unit>
       <trans-unit id="24652e28b94087f3d087bba10c9603848262cafa" resname="Blood">
         <source>Blood</source>
@@ -279,7 +279,7 @@
       </trans-unit>
       <trans-unit id="42b657c8bebf02e57f682d03428fbdc6bcae2837" resname="Blood culture broth">
         <source>Blood culture broth</source>
-        <target>Blood culture broth</target>
+        <target>Caldo de cultivo de sangre</target>
       </trans-unit>
       <trans-unit id="0504fc68c397e6f1d5a2d8342b06e545a890d772" resname="Bloody">
         <source>Bloody</source>
@@ -287,23 +287,22 @@
       </trans-unit>
       <trans-unit id="c70830a91bab7a8b117d115cb089d45a9663de74" resname="Both (ORS/ORT and IV fluids)">
         <source>Both (ORS/ORT and IV fluids)</source>
-        <target>Ambos (
-Both (ORS/ORT and IV fluids)</target>
+        <target>Ambos (ORS/ORT y líquidos intravenosos)</target>
       </trans-unit>
       <trans-unit id="d5b8ecfb0f8c2745aa1d63559a0e837f57b8df70" resname="By Country">
         <source>By Country</source>
-        <target>By Country</target>
+        <target>Por pais (año hasta la fecha)</target>
         <jms:reference-file line="48">views/Security/homepage.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="32aed76fc277287cf218e00c2786cfeee22c813f" resname="By Diagnosis">
         <source>By Diagnosis</source>
-        <target>Por Diagnóstico</target>
+        <target>Por diagnóstico (Año hasta la fecha)</target>
         <jms:reference-file line="24">views/Security/homepage.html.twig</jms:reference-file>
         <jms:reference-file line="2">views/Report/byDiagnosisGraph.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="32e1bbe2f5c8a9c4483c352b659be1e0f7d9c46a" resname="By Site">
         <source>By Site</source>
-        <target>Por Sitio</target>
+        <target>Por Sitio (Año hasta la fecha)</target>
         <jms:reference-file line="70">views/Security/homepage.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="32096c2e0eff33d844ee6d675407ace18289357d" resname="C">
@@ -312,12 +311,12 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="26a272d368f7a0ded2093637af1121af91bae4b3" resname="CSF">
         <source>CSF</source>
-        <target>LCR</target>
+        <target>CSF (LCR)</target>
         <jms:reference-file line="19">views/IBD/editLab.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="075a704274c71eb4f91f3b8d6e34f3fea19765f9" resname="CSF + Isolate">
         <source>CSF + Isolate</source>
-        <target>CSF + Aislar</target>
+        <target>CSF (LCR) + Aislamiento</target>
       </trans-unit>
       <trans-unit id="775275cfc23d95b3976ef3bbea43f42a663ffdda" resname="CT Values From PCR">
         <source>CT Values From PCR</source>
@@ -349,7 +348,7 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="152bf989ec80730f6a4c6891be9103fd67dbd641" resname="Case Based Demographics">
         <source>Case Based Demographics</source>
-        <target>Identificación</target>
+        <target>Demografía </target>
         <jms:reference-file line="18">views/RotaVirus/edit.html.twig</jms:reference-file>
         <jms:reference-file line="12">views/RotaVirus/show.html.twig</jms:reference-file>
         <jms:reference-file line="18">views/IBD/edit.html.twig</jms:reference-file>
@@ -357,7 +356,7 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="a2c77e8741b8bda3439983fd14105607d2e54830" resname="Case ID">
         <source>Case ID</source>
-        <target>Sistema ID</target>
+        <target>Identificación de sistema</target>
         <jms:reference-file line="4">views/Case/region.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="589acc5d25d7be341d54af342e812f3cc85865e6" resname="Case Progression">
@@ -463,7 +462,7 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="5ebca3a6a2449f0bf89505b4220616f518244b2c" resname="Csf Pcr">
         <source>Csf Pcr</source>
-        <target state="new">Csf Pcr</target>
+        <target>Csf Pcr</target>
         <jms:reference-file line="27">views/Report/fieldPopulation.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="daca76f341170488ecbef5e895d310588588cea6" resname="Csf spn pcr positive">
@@ -473,7 +472,7 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="71f440a1dd4cf93fe583dd09db3c514d1b6820c4" resname="Culture Positive and Additional Lab Confirmed Cases By Diagnostic Test">
         <source>Culture Positive and Additional Lab Confirmed Cases By Diagnostic Test</source>
-        <target>Culture Positive and Additional Lab Confirmed Cases By Diagnostic Test</target>
+        <target>Cultivo positivo y casos confirmados por laboratorio por medio de pruebas diagnósticas </target>
         <jms:reference-file line="10">views/Security/homepage.html.twig</jms:reference-file>
         <jms:reference-file line="2">views/Report/culturePositive.html.twig</jms:reference-file>
       </trans-unit>
@@ -511,15 +510,15 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="f0750a96a5015fa6ef305bab6c098c7570710f30" resname="Discharged alive">
         <source>Discharged alive</source>
-        <target>Egreso vivo</target>
+        <target>Egresó vivo</target>
       </trans-unit>
       <trans-unit id="7a67829c1eb32d51c5363f10a1648fc0f87ac83a" resname="Discharged alive, with sequelae">
         <source>Discharged alive, with sequelae</source>
-        <target>Egreso vivo, con secuelas</target>
+        <target>Egresó vivo, con secuelas</target>
       </trans-unit>
       <trans-unit id="e7f4b886ce05008c4c19524028bd23a5d563077b" resname="Discharged alive, without sequelae">
         <source>Discharged alive, without sequelae</source>
-        <target>Egreso vivo, sin secuelas</target>
+        <target>Egresó vivo, sin secuelas</target>
       </trans-unit>
       <trans-unit id="9e9cf3221a30246219863f1d2366e36cb580debc" resname="Documentation">
         <source>Documentation</source>
@@ -537,7 +536,7 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="8fc5633aaf9deadef041285a13d1932225e44e7a" resname="Edit %type% For Case %id%">
         <source>Edit %type% For Case %id%</source>
-        <target>Editar %type% por cada caso %id%</target>
+        <target>Editar %tipo% por caso %id%</target>
         <jms:reference-file line="4">views/RotaVirus/editBaseLab.html.twig</jms:reference-file>
         <jms:reference-file line="4">views/IBD/editBaseLab.html.twig</jms:reference-file>
       </trans-unit>
@@ -549,7 +548,7 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="a0a1daa8eb0790ad70122d43757be2f64b4b5a15" resname="Edit Case Outcome %id%">
         <source>Edit Case Outcome %id%</source>
-        <target>Editar resultado para el Caso %id%</target>
+        <target>Editar resultado del caso %id%</target>
         <jms:reference-file line="4">views/RotaVirus/editOutcome.html.twig</jms:reference-file>
         <jms:reference-file line="4">views/IBD/editOutcome.html.twig</jms:reference-file>
       </trans-unit>
@@ -566,7 +565,7 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="eede5d359286bb963ca85c2cdf35c36ca4e63945" resname="Email / Username">
         <source>Email / Username</source>
-        <target>Email / Nombre de usuario</target>
+        <target>Correo electrónico  / Nombre de usuario</target>
         <jms:reference-file line="31">views/Security/forgotPassword.html.twig</jms:reference-file>
         <jms:reference-file line="32">views/Security/login.html.twig</jms:reference-file>
       </trans-unit>
@@ -660,33 +659,33 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="83b5ccff29458393b2e256062ff4a7594f40fa76" resname="Genotyping">
         <source>Genotyping</source>
-        <target>Proceso de genotipo</target>
+        <target>Genotipificación</target>
         <jms:reference-file line="19">views/RotaVirus/editLab.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="f3dd4543ce161fe06cf1ba8fcc0bff7d6b1370a8" resname="Genotyping Results">
         <source>Genotyping Results</source>
-        <target>Resultados de genotipos</target>
+        <target>Resultados de genotipificación</target>
         <jms:reference-file line="20">views/RotaVirus/editBaseLab.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="4ff0ff91948f32f76ba7932b7290ac166d1cc17c" resname="Gm neg coccobacilli">
         <source>Gm neg coccobacilli</source>
-        <target>Coccobacillus Gram neg</target>
+        <target>Cocobacilos Gram negativo</target>
       </trans-unit>
       <trans-unit id="1767903220844ef5c8daca4f09c1f9a9095e66eb" resname="Gm neg diplococci">
         <source>Gm neg diplococci</source>
-        <target>Diplococcus Gram neg</target>
+        <target>Diplococos Gram negativo</target>
       </trans-unit>
       <trans-unit id="c3811b853d3bbc1dbc0f48bfdee8c53676d10cf1" resname="Gm neg rods">
         <source>Gm neg rods</source>
-        <target>Bacillus Gram neg</target>
+        <target>Bacilos Gram negativo </target>
       </trans-unit>
       <trans-unit id="f9506238d56b08069e0848e4743f1986574c5d15" resname="Gm pos cocci clusters">
         <source>Gm pos cocci clusters</source>
-        <target>Coccus en racimos Gram pos</target>
+        <target>Cocos en agrupaciones Gram positivo</target>
       </trans-unit>
       <trans-unit id="4d766e505de0a11fa04948d0e408049dd9cf70ef" resname="Gm pos cocci pairs">
         <source>Gm pos cocci pairs</source>
-        <target>Diplococcus Gram pos</target>
+        <target>Cocos en pares Gram positivo</target>
       </trans-unit>
       <trans-unit id="982b1ab0cde80096b718ed4035853762de20c0ec" resname="Gram-negative organism">
         <source>Gram-negative organism</source>
@@ -775,18 +774,18 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="d50bc05be0e9b0f49860413c3121d65e4112d904" resname="Lab Information">
         <source>Lab Information</source>
-        <target>Laboratorio de Información</target>
+        <target>Información de Laboratorio</target>
         <jms:reference-file line="19">views/RotaVirus/editBaseLab.html.twig</jms:reference-file>
         <jms:reference-file line="18">views/IBD/editBaseLab.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="18f17c05e2c86b15dc0c6b17fe2b8d184faaa33e" resname="Lab Other Fluid">
         <source>Lab Other Fluid</source>
-        <target>Laboratorio de otros fluidos</target>
+        <target>Otros fluidos de Laboratorio </target>
         <jms:reference-file line="24">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="e3f535c7d82c4f97e1e3d09c16cfa3015b88d24b" resname="Lab Serotype">
         <source>Lab Serotype</source>
-        <target>Laboratorio de serotipo</target>
+        <target>Serotipos de Laboratorio </target>
         <jms:reference-file line="25">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="955eabd1c90a1eb745aabc1978cd51b11990901d" resname="Lab-confirmed for HI">
@@ -822,7 +821,7 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="dfb450c7e95da7c8d07b85a3f8e61d398ff68681" resname="Latest Rotavirus Cases">
         <source>Latest Rotavirus Cases</source>
-        <target>Últimos casos de Rotavirus</target>
+        <target>Casos más recientes de Rotavirus</target>
         <jms:reference-file line="10">views/RotaVirus/index.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="40032b68c83b0f50859aca85848f028fa4b91603" resname="Left/Discharged against medical advice">
@@ -831,7 +830,7 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="61d14bfd8db9302c47a91363bfc3c215f747d391" resname="Lists the last %count% IBD cases">
         <source>Lists the last %count% IBD cases</source>
-        <target>Lista de últimos casos de IBD %count%</target>
+        <target>Lista de últimos casos de IBD %cantidad%</target>
         <jms:reference-file line="3">views/IBD/index.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="6f604eed838cef726bbc73efd2a61ab5d6b7bdd4" resname="Lists the last %count% rotavirus cases">
@@ -850,7 +849,7 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="d6de9f59c2cb1369385079580de553a940a1bf49" resname="MenAfriVac (conjugate MenA)">
         <source>MenAfriVac (conjugate MenA)</source>
-        <target>MenAfriVac (conjugate MenA)</target>
+        <target>MenAfriVac (conjugada MenA)</target>
       </trans-unit>
       <trans-unit id="f16832de90bb82ee2e0c478d12b7006f7099aef4" resname="Meningitis">
         <source>Meningitis</source>
@@ -869,21 +868,21 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="23c675e43706c4c645825a7c3921a55029f1c1ba" resname="Multiple (i.e. Meningitis and/or Pneumonia and/or Sepsis)">
         <source>Multiple (i.e. Meningitis and/or Pneumonia and/or Sepsis)</source>
-        <target>Múltiple (ej: la meningitis y/o neumonía)</target>
+        <target>Múltiple (ej: la meningitis y/o neumonía y/o septicemia)</target>
       </trans-unit>
       <trans-unit id="4c9d813165b8a1f4f350c44565ac62a003f5b4c4" resname="Multiple (i.e. suspected meningitis and/or pneumonia and/or sepsis)">
         <source>Multiple (i.e. suspected meningitis and/or pneumonia and/or sepsis)</source>
-        <target>Múltiple (ej: sospecha de meningitis y/o neumonía)</target>
+        <target>Múltiple (ej: sospecha de meningitis y/o neumonía y/o septicemia )</target>
       </trans-unit>
       <trans-unit id="a59ba24d63e9c67fe81bbc277e1424777efe1eda" resname="NL">
         <source>NL</source>
-        <target>LN</target>
+        <target>NL</target>
         <jms:reference-file line="64">SentinelBundle/Twig/CaseActions.php</jms:reference-file>
         <jms:reference-file line="99">SentinelBundle/Twig/CaseActions.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="709a23220f2c3d64d1e1d6d18c4d5280f8d82fca" resname="Name">
         <source>Name</source>
-        <target>Nombre</target>
+        <target>Nombres</target>
         <jms:reference-file line="10">views/User/profile.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="727d00eb2ea7df33a9b7f2a6ab4bb65ec77db833" resname="National Lab">
@@ -1102,7 +1101,7 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="e3b21a540a718b7284f94d7105bb256a826d03ca" resname="Please Provide Your Email Address">
         <source>Please Provide Your Email Address</source>
-        <target>Por favor provee su correo electrónico</target>
+        <target>Por favor provea su correo electrónico</target>
         <jms:reference-file line="25">views/Security/forgotPassword.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="a7942f226ed1080246f2b2fde35b11e19a91e6a8" resname="Please Select...">
@@ -1176,7 +1175,7 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="3406b0a2aba3d78143f9395f10b3aec35a209650" resname="Region API">
         <source>Region API</source>
-        <target state="new">Region API</target>
+        <target>Región API</target>
       </trans-unit>
       <trans-unit id="b846d1c08e6f20ab3876a6108664a6e90d0e469d" resname="Region Import/Export">
         <source>Region Import/Export</source>
@@ -1256,14 +1255,14 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="5b6825d3c67a35898b7b65fd986a7fe6ca47771d" resname="Site">
         <source>Site</source>
-        <target>Sitio</target>
+        <target>Sitio / Lugar</target>
         <jms:reference-file line="16">views/Report/fieldPopulation.html.twig</jms:reference-file>
         <jms:reference-file line="7">views/Case/region.html.twig</jms:reference-file>
         <jms:reference-file line="6">views/Case/country.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="799b6553db51482380a99f5129803abdccded031" resname="Site API">
         <source>Site API</source>
-        <target>Sitio API</target>
+        <target>Sitio/Lugar API</target>
       </trans-unit>
       <trans-unit id="baaabfc71e818fe50aefacb38b9f9a6beb219123" resname="Site Case ID">
         <source>Site Case ID</source>
@@ -1274,7 +1273,7 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="501fbc593e79084b78bc093c5b3331e7303a780c" resname="Site Import/Export">
         <source>Site Import/Export</source>
-        <target>Sitio de Importar/Exportar</target>
+        <target>Sitio Importar/Exportar</target>
       </trans-unit>
       <trans-unit id="f70c03081c5209cc4aa55b21d6f3911aa88a33e0" resname="Site Lab">
         <source>Site Lab</source>
@@ -1286,7 +1285,7 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="8b679a66a85377a66234e35eb7c30900f0005b4b" resname="Specimen Collection Data">
         <source>Specimen Collection Data</source>
-        <target>Datos de recolección de muestra</target>
+        <target>Datos de colección de espécimen </target>
         <jms:reference-file line="21">views/RotaVirus/edit.html.twig</jms:reference-file>
         <jms:reference-file line="15">views/RotaVirus/show.html.twig</jms:reference-file>
         <jms:reference-file line="23">views/IBD/edit.html.twig</jms:reference-file>
@@ -1307,7 +1306,7 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="bae7d5be70820ed56467bd9a63744e23b47bd711" resname="Status">
         <source>Status</source>
-        <target>Situación</target>
+        <target>Condición </target>
         <jms:reference-file line="9">views/Case/region.html.twig</jms:reference-file>
         <jms:reference-file line="8">views/Case/site.html.twig</jms:reference-file>
         <jms:reference-file line="7">views/Case/country.html.twig</jms:reference-file>
@@ -1318,19 +1317,19 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="e2cbe6c8e35a0c06ef807e763165a5ca499b6237" resname="Suspected">
         <source>Suspected</source>
-        <target>Sospechoso</target>
+        <target>Sospecha</target>
       </trans-unit>
       <trans-unit id="610e54193c6ec1d6efb3c43882c090ea0d779ae6" resname="Suspected meningitis">
         <source>Suspected meningitis</source>
-        <target>Sospechoso de meningitis</target>
+        <target>Sospecha de meningitis</target>
       </trans-unit>
       <trans-unit id="69cd47ee3aa8386c2319f9c2c3ae298350c4c0eb" resname="Suspected pneumonia">
         <source>Suspected pneumonia</source>
-        <target>Sospechoso de neumonía</target>
+        <target>Sospecha de neumonía</target>
       </trans-unit>
       <trans-unit id="7b18d85ccaae50c2f88c2e6b9222d29547ece832" resname="Suspected sepsis">
         <source>Suspected sepsis</source>
-        <target>Sospecha clínica de Septicemia</target>
+        <target>Sospecha de septicemia</target>
       </trans-unit>
       <trans-unit id="3c4e7e374b8f093a9175523a9788473c2904dfdb" resname="Suspected severe pneumonia">
         <source>Suspected severe pneumonia</source>
@@ -1348,7 +1347,7 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="831377cf9256094e86e41449c9a0b0d1160af443" resname="System Id">
         <source>System Id</source>
-        <target>System Id</target>
+        <target>ID del sistema</target>
         <jms:reference-file line="42">views/RotaVirus/edit.html.twig</jms:reference-file>
         <jms:reference-file line="46">views/IBD/edit.html.twig</jms:reference-file>
       </trans-unit>
@@ -1373,7 +1372,7 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="05f86bc979ed4852564f07f80c6a08b06abbbf6f" resname="Turbid/Cloudy">
         <source>Turbid/Cloudy</source>
-        <target>Turbido / Nublado</target>
+        <target>Turbido  / Nublado</target>
       </trans-unit>
       <trans-unit id="bc7819b34ff87570745fbe461e36a16f80e562ce" resname="Unknown">
         <source>Unknown</source>
@@ -1464,12 +1463,12 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="bd979f0607061e62ae45e30554a9d1ff1c2754d3" resname="admin.form-can-create-national-lab-record">
         <source>admin.form-can-create-national-lab-record</source>
-        <target>Puede Crearse un Laboratorio Nacional de Resultados</target>
+        <target>Puede crearse un Laboratorio Nacional de Resultados</target>
         <jms:reference-file line="99">SentinelBundle/Admin/UserAdmin.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="6ce41a81103332954778b43d4e624fcc694f6c69" resname="admin.form-can-create-reference-lab-record">
         <source>admin.form-can-create-reference-lab-record</source>
-        <target>Puede Crearse un Laboratorio de Can Create Reference Lab Resultsadmin.form-can-create-reference-lab-record</target>
+        <target>Puede crearse un Laboratorio de Referencia de Resultados</target>
         <jms:reference-file line="98">SentinelBundle/Admin/UserAdmin.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="83dc047b895b5a3ea8064c71b7bea957ee38c40e" resname="admin.form-can-create-sitelab-record">
@@ -1479,7 +1478,7 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="f45037b362769913c647f9bd5f755b01f22d2120" resname="admin.form-reference-lab">
         <source>admin.form-reference-lab</source>
-        <target>admin.forma-laboratorio-referencia</target>
+        <target>Laboratorio de Referencia</target>
         <jms:reference-file line="100">SentinelBundle/Admin/UserAdmin.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="e9d71f5ee7c92d6dc9e92ffdad17b8bd49418f98" resname="b">
@@ -1638,7 +1637,7 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="e112042fbc3747ae7f364fcef4d5cc4341558bba" resname="ibd-form.blood-collected">
         <source>ibd-form.blood-collected</source>
-        <target>ibd-form.blood-collected</target>
+        <target>¿Se colección sangre del caso?</target>
         <jms:reference-file line="206">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="109">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
@@ -1799,7 +1798,7 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="5bd6a0e287156c72e1ca281ce63053256252457c" resname="ibd-form.csf-lab-datetime">
         <source>ibd-form.csf-lab-datetime</source>
-        <target state="new">ibd-form.csf-lab-datetime</target>
+        <target>Fecha y hora cuando el CSF se recibió en el laboratorio</target>
         <jms:reference-file line="40">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="09822becef86dd896f55f4132c0039831a11910e" resname="ibd-form.csf-lat-done">
@@ -1810,136 +1809,136 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="e7a630fa4e6031cbbe8cb8dfd39f103289148eb0" resname="ibd-form.csf-lat-other">
         <source>ibd-form.csf-lat-other</source>
-        <target state="new">ibd-form.csf-lat-other</target>
+        <target>Si otros organismos fueron identificados in LAT, por favor especifique</target>
         <jms:reference-file line="75">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="48051174eeb9db24836a6d415350864dd0fb3027" resname="ibd-form.csf-lat-result">
         <source>ibd-form.csf-lat-result</source>
-        <target state="new">ibd-form.csf-lat-result</target>
+        <target>Resultados de la prueba de aglutinación en latex (LAT)</target>
         <jms:reference-file line="73">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="62979686cc66be6b7d5e0ae6cdb418c262112eb8" resname="ibd-form.csf-pcr-done">
         <source>ibd-form.csf-pcr-done</source>
-        <target state="new">ibd-form.csf-pcr-done</target>
+        <target>¿Se hizo el PCR en el CSF?</target>
         <jms:reference-file line="224">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="77">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="0d771aab23a8ae85f2f333ef19605a7dd566bd4a" resname="ibd-form.csf-pcr-other">
         <source>ibd-form.csf-pcr-other</source>
-        <target state="new">ibd-form.csf-pcr-other</target>
+        <target>Si otros organismos fueron identificados en el PCR, por favor especifique </target>
         <jms:reference-file line="82">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="34df514064a1c99a4be13c40f837c8699b111d97" resname="ibd-form.csf-pcr-result">
         <source>ibd-form.csf-pcr-result</source>
-        <target state="new">ibd-form.csf-pcr-result</target>
+        <target>Resultados del examen de PCR del CSF</target>
         <jms:reference-file line="79">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="c9ed952e371bae224348f3f066f85d0f41092a2e" resname="ibd-form.csf-protein">
         <source>ibd-form.csf-protein</source>
-        <target state="new">ibd-form.csf-protein</target>
+        <target>Proteína medida del CSF</target>
         <jms:reference-file line="219">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="44">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="6f3302e375553355129bf6ddd6e5e3ac374d6783" resname="ibd-form.csf-store">
         <source>ibd-form.csf-store</source>
-        <target state="new">ibd-form.csf-store</target>
+        <target>El laboratorio nacional  del sitio Sentinel, ¿conservó la cantidad restante del CSF? </target>
         <jms:reference-file line="226">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="84">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="c1a544dc9a388f31f5139dc7810ccd9cd26f2d3d" resname="ibd-form.csf-wcc">
         <source>ibd-form.csf-wcc</source>
-        <target state="new">ibd-form.csf-wcc</target>
+        <target>Recuento de glóbulos blancos en el CSF</target>
         <jms:reference-file line="217">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="42">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="1df7f169999de566518bcc5ce880d14ff589a65e" resname="ibd-form.csfDateTime">
         <source>ibd-form.csfDateTime</source>
-        <target state="new">ibd-form.csfDateTime</target>
+        <target>Fecha y hora cuando se coleccionó el CSF  (Ej. punción lumbar)</target>
         <jms:reference-file line="216">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="c92accbe22abf53acac506dcede212ab0b061bca" resname="ibd-form.csfSiteId">
         <source>ibd-form.csfSiteId</source>
-        <target state="new">ibd-form.csfSiteId</target>
+        <target>Identificación del espécimen CSF</target>
         <jms:reference-file line="215">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="3f1d425f0ca33eb263656cd38f81245df12652d3" resname="ibd-form.cxr-additional-result">
         <source>ibd-form.cxr-additional-result</source>
-        <target state="new">ibd-form.cxr-additional-result</target>
+        <target>Resultado adicional de rayos X</target>
         <jms:reference-file line="187">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="169">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="15177e1c56b680c8d5a87969ec75f40977e4d323" resname="ibd-form.cxr-done">
         <source>ibd-form.cxr-done</source>
-        <target state="new">ibd-form.cxr-done</target>
+        <target>ibd-form.cxr-done</target>
         <jms:reference-file line="185">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="164">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="a4e839aa28b58c472aa65b7799d40bd0fd471511" resname="ibd-form.cxr-result">
         <source>ibd-form.cxr-result</source>
-        <target state="new">ibd-form.cxr-result</target>
+        <target>Resultado de los rayos X del pecho </target>
         <jms:reference-file line="186">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="166">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="f5b41b5ab6d8daab5f7fe41f3c3fcfe9d5c5e7a6" resname="ibd-form.date-of-birth">
         <source>ibd-form.date-of-birth</source>
-        <target state="new">ibd-form.date-of-birth</target>
+        <target>Fecha de nacimiento </target>
         <jms:reference-file line="47">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="cf3bbeed2e41694d4cc7049cbc8c8afa5d89f01e" resname="ibd-form.date-of-birth-known">
         <source>ibd-form.date-of-birth-known</source>
-        <target state="new">ibd-form.date-of-birth-known</target>
+        <target>¿Se conoce la fecha de nacimiento? </target>
         <jms:reference-file line="45">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="3a97aff1bba318c7ba48e7b474619e12a0a6f5e7" resname="ibd-form.date-of-birth-months">
         <source>ibd-form.date-of-birth-months</source>
-        <target state="new">ibd-form.date-of-birth-months</target>
+        <target>Fecha de nacimiento en meses </target>
         <jms:reference-file line="52">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="6697feb3145f37df49ce8507aa55d9e426c0db00" resname="ibd-form.date-of-birth-years">
         <source>ibd-form.date-of-birth-years</source>
-        <target state="new">ibd-form.date-of-birth-years</target>
+        <target>Fecha de nacimiento en años </target>
         <jms:reference-file line="50">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="8760ea4199d8af55a9495e3c8121afbfb6e37d2a" resname="ibd-form.discharge-class">
         <source>ibd-form.discharge-class</source>
-        <target state="new">ibd-form.discharge-class</target>
+        <target>Clasificación definitiva del caso </target>
         <jms:reference-file line="32">Form/IBD/OutcomeType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="43b742aff24ad27b9663f69a87c048a32a7677a8" resname="ibd-form.discharge-diagnosis">
         <source>ibd-form.discharge-diagnosis</source>
-        <target state="new">ibd-form.discharge-diagnosis</target>
+        <target>Diagnostico Medico cuando se dio de alta</target>
         <jms:reference-file line="27">Form/IBD/OutcomeType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="881adb5956f9bab1e8299d53b87a47af43066c64" resname="ibd-form.discharge-diagnosis-other">
         <source>ibd-form.discharge-diagnosis-other</source>
-        <target state="new">ibd-form.discharge-diagnosis-other</target>
+        <target>Si el diagnostico dado cuando se dio de alta is "otra", por favor especifique </target>
         <jms:reference-file line="29">Form/IBD/OutcomeType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="449eca1068df8c1eb5acfc2604630209c4806be8" resname="ibd-form.discharge-outcome">
         <source>ibd-form.discharge-outcome</source>
-        <target state="new">ibd-form.discharge-outcome</target>
+        <target>Resultado del caso cuando se dio de alta</target>
         <jms:reference-file line="26">Form/IBD/OutcomeType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="652eccdd58ea42f992238feca99377f59268cec5" resname="ibd-form.district">
         <source>ibd-form.district</source>
-        <target state="new">ibd-form.district</target>
+        <target>Distrito</target>
         <jms:reference-file line="55">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="e4a2e1a6b298011a13a7c5fde60260b928733c6f" resname="ibd-form.dob">
         <source>ibd-form.dob</source>
-        <target state="new">ibd-form.dob</target>
+        <target>Fecha de nacimiento</target>
         <jms:reference-file line="46">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="53fc870b0397c0effd924b455bfb6dd9d1ea165b" resname="ibd-form.first-name">
         <source>ibd-form.first-name</source>
-        <target state="new">ibd-form.first-name</target>
+        <target>Nombres</target>
         <jms:reference-file line="46">views/RotaVirus/show.html.twig</jms:reference-file>
         <jms:reference-file line="58">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="43">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="042a6d61c36ae7348e0cdadef47bf027c0d8d0e0" resname="ibd-form.gender">
         <source>ibd-form.gender</source>
-        <target state="new">ibd-form.gender</target>
+        <target>Sexo</target>
         <jms:reference-file line="54">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="54">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
@@ -1951,53 +1950,55 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="0da45bd56fa56862eebf9da5bebb9c33d58b4081" resname="ibd-form.hi-serotype-other">
         <source>ibd-form.hi-serotype-other</source>
-        <target state="new">ibd-form.hi-serotype-other</target>
+        <target>Influenza serotipo es o no es por PCR  
+
+</target>
         <jms:reference-file line="92">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="49d1ec463de133d2d2ecb102a8f7f16395fbb0a3" resname="ibd-form.hib-doses">
         <source>ibd-form.hib-doses</source>
-        <target state="new">ibd-form.hib-doses</target>
+        <target>Si la respuesta es sí, indique el número de dosis de vacunas que contienen Hib   </target>
         <jms:reference-file line="83">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="79">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="35b71f69793e7ce665cc62c1b4f69665a9d20ee2" resname="ibd-form.hib-most-recent-dose">
         <source>ibd-form.hib-most-recent-dose</source>
-        <target state="new">ibd-form.hib-most-recent-dose</target>
+        <target>ibd-form.hib-most-recent-dose</target>
         <jms:reference-file line="83">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="889252566ba2c10b3e775d697365b8a8381a2597" resname="ibd-form.hib-received">
         <source>ibd-form.hib-received</source>
-        <target state="new">ibd-form.hib-received</target>
+        <target>¿Recibió la vacuna Hib?</target>
         <jms:reference-file line="79">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="78">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="9327cc648f6a6d816787fd228608a00e5e4ec50e" resname="ibd-form.id">
         <source>ibd-form.id</source>
-        <target state="new">ibd-form.id</target>
+        <target>Identificación del sistema </target>
         <jms:reference-file line="38">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="f74e6a891f81b39bd3022ad46a51937d90d20ad3" resname="ibd-form.isol-store">
         <source>ibd-form.isol-store</source>
-        <target state="new">ibd-form.isol-store</target>
+        <target>¿El laboratorio nacional del Sentinel local, conservó el resto de aislamiento?</target>
         <jms:reference-file line="227">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="85">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="221bb596713eb3680a8661a7821f93285464e552" resname="ibd-form.last-name">
         <source>ibd-form.last-name</source>
-        <target state="new">ibd-form.last-name</target>
+        <target>Apellidos </target>
         <jms:reference-file line="50">views/RotaVirus/show.html.twig</jms:reference-file>
         <jms:reference-file line="62">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="42">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="675cb3a0cff157bf7af9c9cc124e5b1f4f7c16fa" resname="ibd-form.men-alt-conscious">
         <source>ibd-form.men-alt-conscious</source>
-        <target state="new">ibd-form.men-alt-conscious</target>
+        <target></target>
         <jms:reference-file line="137">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="68">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="39e71aff4c9bb1d36d83afdd09a89f63c71bddda" resname="ibd-form.men-fever">
         <source>ibd-form.men-fever</source>
-        <target state="new">ibd-form.men-fever</target>
+        <target>¿Existe historia de fiebre?</target>
         <jms:reference-file line="133">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="66">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
@@ -2009,433 +2010,433 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="795c97dad5ffdc4cd53335413bfbb5961dab1dac" resname="ibd-form.men-inability-feed">
         <source>ibd-form.men-inability-feed</source>
-        <target state="new">ibd-form.men-inability-feed</target>
+        <target>¿Tuvo dificultad de beber, mamar o no poder chupar la mama?</target>
         <jms:reference-file line="141">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="70">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="7fb36591a7c5034b63e99f757c5281b0a9b445e0" resname="ibd-form.men-lethargy">
         <source>ibd-form.men-lethargy</source>
-        <target state="new">ibd-form.men-lethargy</target>
+        <target>ibd-form.men-lethargy</target>
         <jms:reference-file line="157">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="76">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="25d8e9216c230be4b56ee8695add622327bbce42" resname="ibd-form.men-neck-stiff">
         <source>ibd-form.men-neck-stiff</source>
-        <target state="new">ibd-form.men-neck-stiff</target>
+        <target>¿Se presentó con rigidez en el cuello?</target>
         <jms:reference-file line="145">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="b47e51722037d4b515b3d3ff6e009c6908552b27" resname="ibd-form.men-rash">
         <source>ibd-form.men-rash</source>
-        <target state="new">ibd-form.men-rash</target>
+        <target>¿Se presentó con petequias/ erupción púrpura? </target>
         <jms:reference-file line="149">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="73">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="fba5837f1590c2922642e33ce6468b70c9ec5804" resname="ibd-form.men-received">
         <source>ibd-form.men-received</source>
-        <target state="new">ibd-form.men-received</target>
+        <target>¿Recibió la vacuna contra el meningococo?</target>
         <jms:reference-file line="99">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="232da5630f9eff618fe936dc4a0492c1f5d5a5f2" resname="ibd-form.men-seizures">
         <source>ibd-form.men-seizures</source>
-        <target state="new">ibd-form.men-seizures</target>
+        <target>¿Se presentó con convulsiones?</target>
         <jms:reference-file line="129">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="65">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="8359d128b473cf32a9ec4984afb5571c1043cbf4" resname="ibd-form.men-stiff-neck">
         <source>ibd-form.men-stiff-neck</source>
-        <target state="new">ibd-form.men-stiff-neck</target>
+        <target>¿Se presentó con rigidez en el cuello?</target>
         <jms:reference-file line="72">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="1d1c64818752598706df6d4e14dbe5b8c776ad5c" resname="ibd-form.men-type">
         <source>ibd-form.men-type</source>
-        <target state="new">ibd-form.men-type</target>
+        <target>Si la respuesta es sí, seleccione el tipo de vacuna que recibió  contra el meningococo </target>
         <jms:reference-file line="99">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="101">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="0209a43a47c9040d1b3548fe4ef3a438fec26cf3" resname="ibd-form.mening-received">
         <source>ibd-form.mening-received</source>
-        <target state="new">ibd-form.mening-received</target>
+        <target>¿Recibió la vacuna contra el meningococo?</target>
         <jms:reference-file line="95">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="e0e54ee64a1605a52d36f3f67073df73a735019f" resname="ibd-form.meningMostRecentDose">
         <source>ibd-form.meningMostRecentDose</source>
-        <target state="new">ibd-form.meningMostRecentDose</target>
+        <target>Si la respuesta es sí, indique la fecha de la vacuna más reciente en contra el meningococo </target>
         <jms:reference-file line="103">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="105">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="da1a97aa53498979e729700a7e7a9323d764fd29" resname="ibd-form.nm-serogroup">
         <source>ibd-form.nm-serogroup</source>
-        <target state="new">ibd-form.nm-serogroup</target>
+        <target>Nm de resultado de Serogrupo </target>
         <jms:reference-file line="256">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="94">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="3d682a7a4fde02d81a3a9723fb28bc250dffbf94" resname="ibd-form.nm-serogroup-other">
         <source>ibd-form.nm-serogroup-other</source>
-        <target state="new">ibd-form.nm-serogroup-other</target>
+        <target>Nm Serogrupo Otro </target>
         <jms:reference-file line="96">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="f2a990a5682e586d6f19820c4587d563a66b2eb1" resname="ibd-form.onset-date">
         <source>ibd-form.onset-date</source>
-        <target state="new">ibd-form.onset-date</target>
+        <target>Fecha de inicio de la enfermedad </target>
         <jms:reference-file line="63">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="6914b1791ddf8fcc5be2232c42dd5fa8a16a3e2d" resname="ibd-form.other-cult-done1">
         <source>ibd-form.other-cult-done1</source>
-        <target state="new">ibd-form.other-cult-done1</target>
+        <target>¿Se hizo cultivo de otros especímenes ? (Ej. líquido pleural)</target>
         <jms:reference-file line="125">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="2237f5d2f7a05e8620bc7e1441b172f8b5d9887a" resname="ibd-form.other-cult-other">
         <source>ibd-form.other-cult-other</source>
-        <target state="new">ibd-form.other-cult-other</target>
+        <target>Si otros organismos fueron identificados en cultivos hechos en otro espécimen, (Ej. líquido pleural) por favor especifique</target>
         <jms:reference-file line="246">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="130">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="1ce1e91fba4569e056c07202acdaec0e42324e32" resname="ibd-form.other-cult-result">
         <source>ibd-form.other-cult-result</source>
-        <target state="new">ibd-form.other-cult-result</target>
+        <target>Resultados de cultivo hechos en otro espécimen (Ej. líquido pleural)</target>
         <jms:reference-file line="245">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="128">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="1908116a6aebd13840d41b1984a4e743fdbab76e" resname="ibd-form.other-culture">
         <source>ibd-form.other-culture</source>
-        <target state="new">ibd-form.other-culture</target>
+        <target>Otro cultivo </target>
         <jms:reference-file line="244">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="200fbe0f4392a0194affd6dada0367aa49ad8166" resname="ibd-form.otherSpecimenCollected">
         <source>ibd-form.otherSpecimenCollected</source>
-        <target state="new">ibd-form.otherSpecimenCollected</target>
+        <target>Otro espécimen coleccionado </target>
         <jms:reference-file line="122">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="67cf1c3078f9ce891b406ec235255eeb5122a1ec" resname="ibd-form.otherSpecimenOther">
         <source>ibd-form.otherSpecimenOther</source>
-        <target state="new">ibd-form.otherSpecimenOther</target>
+        <target>Otro espécimen - otro </target>
         <jms:reference-file line="123">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="a91a868a284701101932408cf82f74c98a512f1c" resname="ibd-form.parental-name">
         <source>ibd-form.parental-name</source>
-        <target state="new">ibd-form.parental-name</target>
+        <target>Nombre de los padres </target>
         <jms:reference-file line="54">views/RotaVirus/show.html.twig</jms:reference-file>
         <jms:reference-file line="66">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="44">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="e654aa73e9d5bf4f0bca9855f4733905bdd0fdb7" resname="ibd-form.pcv-doses">
         <source>ibd-form.pcv-doses</source>
-        <target state="new">ibd-form.pcv-doses</target>
+        <target>Si la respuesta es si, ¿ cuántas dosis de PCV?</target>
         <jms:reference-file line="91">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="88">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="29d00dea515e4b0b5e1edcd9dfd7116988414ace" resname="ibd-form.pcv-most-recent-dose">
         <source>ibd-form.pcv-most-recent-dose</source>
-        <target state="new">ibd-form.pcv-most-recent-dose</target>
+        <target>La más reciente dosis de PCV</target>
         <jms:reference-file line="95">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="edffc7b5bcbc1aae27611591b542bf92fc9def3e" resname="ibd-form.pcv-received">
         <source>ibd-form.pcv-received</source>
-        <target state="new">ibd-form.pcv-received</target>
+        <target>¿Recibió la vacuna contra el PCV?</target>
         <jms:reference-file line="87">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="87">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="835bd9d7717d9f2bc8ffd452e61f16e88e979015" resname="ibd-form.pcv-type">
         <source>ibd-form.pcv-type</source>
-        <target state="new">ibd-form.pcv-type</target>
+        <target>Tipo de PCV</target>
         <jms:reference-file line="91">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="b0449a1ff36c969abd8cf18b1487c88f3a293214" resname="ibd-form.pneu-chest-indraw">
         <source>ibd-form.pneu-chest-indraw</source>
-        <target state="new">ibd-form.pneu-chest-indraw</target>
+        <target>¿Se presento con tiraje pectoral?</target>
         <jms:reference-file line="170">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="149">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="cee97bdbcda81468703ffab178f32adf944044bc" resname="ibd-form.pneu-cough">
         <source>ibd-form.pneu-cough</source>
-        <target state="new">ibd-form.pneu-cough</target>
+        <target>¿Tenía tos?</target>
         <jms:reference-file line="174">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="151">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="371d8ce20eeca8a6d6b8a903b0e7305e7710103d" resname="ibd-form.pneu-cyanosis">
         <source>ibd-form.pneu-cyanosis</source>
-        <target state="new">ibd-form.pneu-cyanosis</target>
+        <target>¿Se presentó con síntomas de cianosis central/coloración azul de la piel?</target>
         <jms:reference-file line="178">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="153">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="5adcdcdfbff65d2302ee31810b9b0dee4a4e5dda" resname="ibd-form.pneu-diff-breathe">
         <source>ibd-form.pneu-diff-breathe</source>
-        <target state="new">ibd-form.pneu-diff-breathe</target>
+        <target>¿Tenía dificultad para respirar?</target>
         <jms:reference-file line="166">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="147">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="96fcdc48070a62f269a74895905465f02bd8664c" resname="ibd-form.pneu-hypothermia">
         <source>ibd-form.pneu-hypothermia</source>
-        <target state="new">ibd-form.pneu-hypothermia</target>
+        <target><![CDATA[¿Se presentó con hipotermia (<36ºC)?]]></target>
         <jms:reference-file line="160">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="92a7812093d74879b99c6e08a3f95d9578c8a24b" resname="ibd-form.pneu-malnutrition">
         <source>ibd-form.pneu-malnutrition</source>
-        <target state="new">ibd-form.pneu-malnutrition</target>
+        <target>¿Se presentó con desnutrición severa?</target>
         <jms:reference-file line="162">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="dfd2e55c201e5e40bbbecd5a0bf9dc02363f3f65" resname="ibd-form.pneu-resp-rate">
         <source>ibd-form.pneu-resp-rate</source>
-        <target state="new">ibd-form.pneu-resp-rate</target>
+        <target>Frecuencia Respiratoria </target>
         <jms:reference-file line="182">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="156">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="ffbe7c62aaf100600d3814e89119ed21efd2db4b" resname="ibd-form.pneu-stridor">
         <source>ibd-form.pneu-stridor</source>
-        <target state="new">ibd-form.pneu-stridor</target>
+        <target>¿Se presentó con estridor aún en estado de calma?</target>
         <jms:reference-file line="155">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="9894592f6ab722886f3b1bc5f430d331b0091f0d" resname="ibd-form.pneu-vomit">
         <source>ibd-form.pneu-vomit</source>
-        <target state="new">ibd-form.pneu-vomit</target>
+        <target>¿Se presentó vomitando todo?</target>
         <jms:reference-file line="158">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="824fb0503c86c2f43cfb72ebc661b81a51aa307e" resname="ibd-form.site">
         <source>ibd-form.site</source>
-        <target state="new">ibd-form.site</target>
+        <target>Lugar</target>
         <jms:reference-file line="48">SentinelBundle/Form/CreateType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="e7f1357adc9d2bf5a62aec9cae6f74fca393f295" resname="ibd-form.spn-serotype">
         <source>ibd-form.spn-serotype</source>
-        <target state="new">ibd-form.spn-serotype</target>
+        <target>SPN Serotipos</target>
         <jms:reference-file line="86">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="10de4d0ee39a98a1579ff9e8af2907e180206158" resname="ibd-form.spn-serotype-other">
         <source>ibd-form.spn-serotype-other</source>
-        <target state="new">ibd-form.spn-serotype-other</target>
+        <target>SPN Serotipos  otros</target>
         <jms:reference-file line="88">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="b906484ba1d128e05a45d71ec5a74cded6c4b8d3" resname="ibd-form.state">
         <source>ibd-form.state</source>
-        <target state="new">ibd-form.state</target>
+        <target>Estado</target>
         <jms:reference-file line="56">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="71f1f4afb087a9789216f0ef7b49adbdf8019595" resname="ibd-rrl-form.bexA">
         <source>ibd-rrl-form.bexA</source>
-        <target state="new">ibd-rrl-form.bexA</target>
+        <target>Resultados del bexA</target>
         <jms:reference-file line="45">Form/IBD/BaseLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="f084bc0e620fff925351a292995462e61b7d10d0" resname="ibd-rrl-form.comment">
         <source>ibd-rrl-form.comment</source>
-        <target state="new">ibd-rrl-form.comment</target>
+        <target>ibd-rrl-form.comment</target>
         <jms:reference-file line="54">Form/IBD/BaseLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="e370d933ed03841f45bac3527ec3477fafc7cc6d" resname="ibd-rrl-form.ctrA">
         <source>ibd-rrl-form.ctrA</source>
-        <target state="new">ibd-rrl-form.ctrA</target>
+        <target>Resultados del ctrA</target>
         <jms:reference-file line="41">Form/IBD/BaseLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="2076b40cb69e6ccb73bd12920c74bcea34d875b6" resname="ibd-rrl-form.date-received">
         <source>ibd-rrl-form.date-received</source>
-        <target state="new">ibd-rrl-form.date-received</target>
+        <target>Fecha que se recibió</target>
         <jms:reference-file line="24">Form/IBD/BaseLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="4431de48fe3914ee5bea2b02bf28656f3609fc5f" resname="ibd-rrl-form.finalResult">
         <source>ibd-rrl-form.finalResult</source>
-        <target state="new">ibd-rrl-form.finalResult</target>
+        <target>Resultado Final</target>
         <jms:reference-file line="47">Form/IBD/BaseLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="2066102b49916398f0130dd7e283fbfcbad72d43" resname="ibd-rrl-form.hiSerotype">
         <source>ibd-rrl-form.hiSerotype</source>
-        <target state="new">ibd-rrl-form.hiSerotype</target>
+        <target>Hi serotipo</target>
         <jms:reference-file line="50">Form/IBD/BaseLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="0cb23a5357bfae65fa556b96a25a25ed39632ab4" resname="ibd-rrl-form.hiSerotype-other">
         <source>ibd-rrl-form.hiSerotype-other</source>
-        <target state="new">ibd-rrl-form.hiSerotype-other</target>
+        <target>Hi serotipo otro</target>
         <jms:reference-file line="255">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="25bdbb8a82fd603598ad8c5de3b0db2299b66a40" resname="ibd-rrl-form.hpd1">
         <source>ibd-rrl-form.hpd1</source>
-        <target state="new">ibd-rrl-form.hpd1</target>
+        <target>Resultados del hpd1</target>
         <jms:reference-file line="43">Form/IBD/BaseLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="adfa4dd800742d1c529d631a49ef387a854fbdd9" resname="ibd-rrl-form.hpd3">
         <source>ibd-rrl-form.hpd3</source>
-        <target state="new">ibd-rrl-form.hpd3</target>
+        <target>Resultados del hpd3</target>
         <jms:reference-file line="44">Form/IBD/BaseLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="f1eb0cb14391d8163de6afa050a9b21f5cc38083" resname="ibd-rrl-form.isolate-type">
         <source>ibd-rrl-form.isolate-type</source>
-        <target state="new">ibd-rrl-form.isolate-type</target>
+        <target>Tipo de aislamiento</target>
         <jms:reference-file line="28">Form/IBD/BaseLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="d43f2018d33e252d8da538b46d2e64bdbd510dc1" resname="ibd-rrl-form.isolate-viable">
         <source>ibd-rrl-form.isolate-viable</source>
-        <target state="new">ibd-rrl-form.isolate-viable</target>
+        <target></target>
         <jms:reference-file line="26">Form/IBD/BaseLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="fd8f2ea74aed7b57ef8f097cfe9b4979bab20774" resname="ibd-rrl-form.lab-id">
         <source>ibd-rrl-form.lab-id</source>
-        <target state="new">ibd-rrl-form.lab-id</target>
+        <target>Identification de Laboratorio</target>
         <jms:reference-file line="21">Form/IBD/BaseLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="3fe0e2ffbe05ae99cc6525b917b8676dc6db73ed" resname="ibd-rrl-form.lytA">
         <source>ibd-rrl-form.lytA</source>
-        <target state="new">ibd-rrl-form.lytA</target>
+        <target>Resultados de lytA</target>
         <jms:reference-file line="40">Form/IBD/BaseLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="f1ea8723ed887f371c5b0f72ec637c767366b181" resname="ibd-rrl-form.nmSerogroup">
         <source>ibd-rrl-form.nmSerogroup</source>
-        <target state="new">ibd-rrl-form.nmSerogroup</target>
+        <target>Nm de serogrupo</target>
         <jms:reference-file line="52">Form/IBD/BaseLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="aaa16f9e6f9a958888061aefee58708928de6b2d" resname="ibd-rrl-form.nmSerogroup-other">
         <source>ibd-rrl-form.nmSerogroup-other</source>
-        <target state="new">ibd-rrl-form.nmSerogroup-other</target>
+        <target>Otro Nm de serogrupo</target>
         <jms:reference-file line="257">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="72c46a15a834a5c83f9d5757e6e8b7f1fa175d23" resname="ibd-rrl-form.pathogen-id-method">
         <source>ibd-rrl-form.pathogen-id-method</source>
-        <target state="new">ibd-rrl-form.pathogen-id-method</target>
+        <target>Métodos usados para la identificación de patógenos </target>
         <jms:reference-file line="30">Form/IBD/BaseLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="0557c33b4eb9b06edb5a039d0b949b9c2420de11" resname="ibd-rrl-form.pathogen-id-other">
         <source>ibd-rrl-form.pathogen-id-other</source>
-        <target state="new">ibd-rrl-form.pathogen-id-other</target>
+        <target>Identificación de patógenos - otro método</target>
         <jms:reference-file line="32">Form/IBD/BaseLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="9995383f100098aee0b4cf066ec2462807012684" resname="ibd-rrl-form.rNasP">
         <source>ibd-rrl-form.rNasP</source>
-        <target state="new">ibd-rrl-form.rNasP</target>
+        <target>Formularios de resultados rNaseP</target>
         <jms:reference-file line="46">Form/IBD/BaseLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="9e530f08f9a9d01ac05a4ff86de08a2191847c48" resname="ibd-rrl-form.sample-type">
         <source>ibd-rrl-form.sample-type</source>
-        <target state="new">ibd-rrl-form.sample-type</target>
+        <target>Tipo de muestra</target>
         <jms:reference-file line="22">Form/IBD/BaseLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="377c797956e51449853a82e29e9bbce404eb4265" resname="ibd-rrl-form.serotype-id-method">
         <source>ibd-rrl-form.serotype-id-method</source>
-        <target state="new">ibd-rrl-form.serotype-id-method</target>
+        <target>Métodos de identification de serotipos</target>
         <jms:reference-file line="35">Form/IBD/BaseLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="738f5dd02c952d044a1711c49db383b9edef721d" resname="ibd-rrl-form.serotype-id-other">
         <source>ibd-rrl-form.serotype-id-other</source>
-        <target state="new">ibd-rrl-form.serotype-id-other</target>
+        <target>Otros métodos de identificación de serotipos</target>
         <jms:reference-file line="37">Form/IBD/BaseLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="9ef8137e76eedd033efa430a81da58095b0bfe6f" resname="ibd-rrl-form.sodC">
         <source>ibd-rrl-form.sodC</source>
-        <target state="new">ibd-rrl-form.sodC</target>
+        <target>Resultados de sodC</target>
         <jms:reference-file line="42">Form/IBD/BaseLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="dd0f50fb8281e71584c12286064afe7a86598675" resname="ibd-rrl-form.spnSerotype">
         <source>ibd-rrl-form.spnSerotype</source>
-        <target state="new">ibd-rrl-form.spnSerotype</target>
+        <target>Serotipos de Spn</target>
         <jms:reference-file line="252">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="48">Form/IBD/BaseLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="dc1b2fd14076d493005b4fce36912de890c073e1" resname="ibd-rrl-form.spnSerotype-other">
         <source>ibd-rrl-form.spnSerotype-other</source>
-        <target state="new">ibd-rrl-form.spnSerotype-other</target>
+        <target>Otros serotipos de Spn</target>
         <jms:reference-file line="253">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="61289dcc293d6f02ff71a837bb4581d20d1316f2" resname="ibd.bexA">
         <source>ibd.bexA</source>
-        <target state="new">ibd.bexA</target>
+        <target>bexA</target>
         <jms:reference-file line="291">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="325">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="7b50728b4f1d1b8a67a7908e3baa16d62473af8e" resname="ibd.ctrA">
         <source>ibd.ctrA</source>
-        <target state="new">ibd.ctrA</target>
+        <target>ctrA</target>
         <jms:reference-file line="287">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="321">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="b61a460df5a5230752bd3907abd9b6cbb670f6d4" resname="ibd.dateReceived">
         <source>ibd.dateReceived</source>
-        <target state="new">ibd.dateReceived</target>
+        <target></target>
         <jms:reference-file line="279">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="313">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="75c24750f2ea2fd63e02979c8e1a10a4046ed122" resname="ibd.hiSerotype">
         <source>ibd.hiSerotype</source>
-        <target state="new">ibd.hiSerotype</target>
+        <target>hi Serotype</target>
         <jms:reference-file line="294">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="328">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="0db7c9647874f109b059f6b651750b600d606026" resname="ibd.hpd1">
         <source>ibd.hpd1</source>
-        <target state="new">ibd.hpd1</target>
+        <target>ibd.hpd1</target>
         <jms:reference-file line="289">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="323">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="9b4df6fb6dd0179349ae6c86df0ca0bfa6f708f5" resname="ibd.hpd3">
         <source>ibd.hpd3</source>
-        <target state="new">ibd.hpd3</target>
+        <target>hpd3</target>
         <jms:reference-file line="290">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="324">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="fc8a4a8e6e9e4d38b2214af83f95f74ca4c72194" resname="ibd.isolateType">
         <source>ibd.isolateType</source>
-        <target state="new">ibd.isolateType</target>
+        <target>Tipo de aislamiento</target>
         <jms:reference-file line="281">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="315">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="63fe34e1c4936ca7d094647ec42ade02a6a08482" resname="ibd.isolateViable">
         <source>ibd.isolateViable</source>
-        <target state="new">ibd.isolateViable</target>
+        <target>ibd.isolateViable</target>
         <jms:reference-file line="280">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="314">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="d229fd5c48c36bce8d007a0dab2808643f24e316" resname="ibd.lytA">
         <source>ibd.lytA</source>
-        <target state="new">ibd.lytA</target>
+        <target>lytA</target>
         <jms:reference-file line="286">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="320">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="d4714f4f356125d89d3d29c094f730b7a82fa414" resname="ibd.nmSerogroup">
         <source>ibd.nmSerogroup</source>
-        <target state="new">ibd.nmSerogroup</target>
+        <target>Numero de resultados de serogrupos</target>
         <jms:reference-file line="295">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="329">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="ecdd9e5aef58c2090d94b2da337e4075230e92a5" resname="ibd.pathogenIdentifierMethod">
         <source>ibd.pathogenIdentifierMethod</source>
-        <target state="new">ibd.pathogenIdentifierMethod</target>
+        <target>ibd.pathogenIdentifierMethod</target>
         <jms:reference-file line="282">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="316">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="78c3ea1f4048bf6a83056df86cee076e667869c7" resname="ibd.pathogenIdentifierOther">
         <source>ibd.pathogenIdentifierOther</source>
-        <target state="new">ibd.pathogenIdentifierOther</target>
+        <target>Otro identificador de patógenos</target>
         <jms:reference-file line="283">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="317">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="b1ee9213cdfc0edaa8e803ff95618b0740b74e21" resname="ibd.rNaseP">
         <source>ibd.rNaseP</source>
-        <target state="new">ibd.rNaseP</target>
+        <target>rNaseP</target>
         <jms:reference-file line="292">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="326">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="76f38fa483effa06902af12b65e7779378e3197a" resname="ibd.sampleType">
         <source>ibd.sampleType</source>
-        <target state="new">ibd.sampleType</target>
+        <target>Tipo de muestra</target>
         <jms:reference-file line="278">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="312">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="9096fa1171730ffd5cb26e8d62ffb57d444b5d6f" resname="ibd.serotypeIdentifier">
         <source>ibd.serotypeIdentifier</source>
-        <target state="new">ibd.serotypeIdentifier</target>
+        <target>Identificador de serotipos</target>
         <jms:reference-file line="284">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="318">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="5fa4e9222297ea42b6f2a23fcbdb906cd72804ef" resname="ibd.serotypeIdentifierOther">
         <source>ibd.serotypeIdentifierOther</source>
-        <target state="new">ibd.serotypeIdentifierOther</target>
+        <target>Otro identificador de serotipos</target>
         <jms:reference-file line="285">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="319">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="348d6dd2724fecb3c18ecdc8a685e75edf328c6f" resname="ibd.sodC">
         <source>ibd.sodC</source>
-        <target state="new">ibd.sodC</target>
+        <target>sodC</target>
         <jms:reference-file line="288">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="322">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="afc949b4dd71f9655e84e5ee9bbec57836fc2eb3" resname="ibd.spnSerotype">
         <source>ibd.spnSerotype</source>
-        <target state="new">ibd.spnSerotype</target>
+        <target>Spn serotipo</target>
         <jms:reference-file line="293">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="327">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
@@ -2446,14 +2447,14 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="7d0869060d78b94d14029316b8ea3e0149ff37d1" resname="meningitis.has-site-national-lab">
         <source>meningitis.has-site-national-lab</source>
-        <target state="new">meningitis.has-site-national-lab</target>
+        <target>El lugar tiene un laboratorio nacional</target>
         <jms:reference-file line="275">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="298">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="309">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="d5c4b3a7af8d9a1caf675e01415d7c32513ff977" resname="meningitis.has-site-reference-lab">
         <source>meningitis.has-site-reference-lab</source>
-        <target state="new">meningitis.has-site-reference-lab</target>
+        <target>El lugar tiene un laboratorio de referencia</target>
         <jms:reference-file line="332">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="5c6441c54e28f6fe1e8319a1069edb607e328614" resname="menu.data-admin">
@@ -2479,12 +2480,12 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="d32de6c30481b27aa39189778084a038b3fe7d42" resname="menu.data-reports-culture-positive">
         <source>menu.data-reports-culture-positive</source>
-        <target state="new">menu.data-reports-culture-positive</target>
+        <target>Reporte de cultivos positivos</target>
         <jms:reference-file line="48">SentinelBundle/Menu/Builder.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="7957870acc95548d273cdbe96a5e710f52a7669f" resname="menu.data-reports-field-population">
         <source>menu.data-reports-field-population</source>
-        <target state="new">menu.data-reports-field-population</target>
+        <target>Reporte de población de área </target>
         <jms:reference-file line="47">SentinelBundle/Menu/Builder.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="6afede7b4c3bae0d1c32c37dcce812c05e0fe74c" resname="menu.data-reports-percent-enrolled">
@@ -2494,22 +2495,22 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="540cd0fc2fe5af1c6ee1e78f1ef303a5abc43171" resname="menu.export">
         <source>menu.export</source>
-        <target state="new">Exportar</target>
+        <target>Exportación de datos</target>
         <jms:reference-file line="61">SentinelBundle/Menu/Builder.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="a7a9a2039b03a8d5e569588adeae98caa145db3f" resname="menu.ibd">
         <source>menu.ibd</source>
-        <target state="new">IBD</target>
+        <target>IBD</target>
         <jms:reference-file line="38">SentinelBundle/Menu/Builder.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="ce6399d3bacfea49ed276b16f403667a7886d5b7" resname="menu.import">
         <source>menu.import</source>
-        <target state="new">Importar</target>
+        <target>Importación de carpetas de datos </target>
         <jms:reference-file line="60">SentinelBundle/Menu/Builder.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="5a371a391a2d1a3b1d5ae22065144126a5fc6308" resname="menu.import-export">
         <source>menu.import-export</source>
-        <target state="new">Importar/Exportar</target>
+        <target>Importaciones y exportaciones</target>
         <jms:reference-file line="59">SentinelBundle/Menu/Builder.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="c136d5b4518049cb39d5e06e6b8dc16619e7f7af" resname="menu.translation">
@@ -2519,17 +2520,17 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="eb33f39189da6e994faddf9398756c7ca05641e2" resname="report-filter-form.admitted-between">
         <source>report-filter-form.admitted-between</source>
-        <target state="new">report-filter-form.admitted-between</target>
+        <target>Fecha de admisión </target>
         <jms:reference-file line="44">Form/Filters/BaseReportFilterType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="77fc283564e57886245b0776cfc690aa3abc4f86" resname="report-filter-form.created-between">
         <source>report-filter-form.created-between</source>
-        <target state="new">report-filter-form.created-between</target>
+        <target>Fecha de creación de registro</target>
         <jms:reference-file line="45">Form/Filters/BaseReportFilterType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="487ef4638703ce80913ffeb25a51fd0d889a1306" resname="rotavirus-form.adequate">
         <source>rotavirus-form.adequate</source>
-        <target state="new">rotavirus-form.adequate</target>
+        <target>Volumen adecuado</target>
         <jms:reference-file line="200">views/RotaVirus/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="4f86b99da1abe8a75314de0f423ce2e998c50002" resname="rotavirus-form.adm-date">
@@ -2549,7 +2550,7 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="ad0ce05802da286f986e9e2c1dd7d54cb23c12f0" resname="rotavirus-form.case-id">
         <source>rotavirus-form.case-id</source>
-        <target state="new">rotavirus-form.case-id</target>
+        <target>Identificación del caso</target>
         <jms:reference-file line="30">views/RotaVirus/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="e4de6aef31f0eb1fadc7cb6cc066c88f5eb98172" resname="rotavirus-form.caseId">
@@ -2617,37 +2618,37 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="6e237a8d6d77169b0a337e90a5737b45f1a4886c" resname="rotavirus-form.elisaDone">
         <source>rotavirus-form.elisaDone</source>
-        <target state="new">rotavirus-form.elisaDone</target>
+        <target>Se ha hecho la prueba de Elisa?</target>
         <jms:reference-file line="208">views/RotaVirus/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="56278e65697d3dc136cd97add67cac757c23493c" resname="rotavirus-form.elisaExpiryDate">
         <source>rotavirus-form.elisaExpiryDate</source>
-        <target state="new">rotavirus-form.elisaExpiryDate</target>
+        <target>Fecha de vencimiento de la prueba de Elisa</target>
         <jms:reference-file line="224">views/RotaVirus/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="79f8255a575536dc6704efd3a381f552f82c3a7e" resname="rotavirus-form.elisaKit">
         <source>rotavirus-form.elisaKit</source>
-        <target state="new">rotavirus-form.elisaKit</target>
+        <target>Tipo de Elisa</target>
         <jms:reference-file line="212">views/RotaVirus/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="4e73ce1990139403dce5bd07cf9944f85445ec83" resname="rotavirus-form.elisaKitOther">
         <source>rotavirus-form.elisaKitOther</source>
-        <target state="new">rotavirus-form.elisaKitOther</target>
+        <target>Otro tipo de Elisa</target>
         <jms:reference-file line="216">views/RotaVirus/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="3824152003937d30d74857bce2ba75680423cd73" resname="rotavirus-form.elisaLoadNumber">
         <source>rotavirus-form.elisaLoadNumber</source>
-        <target state="new">rotavirus-form.elisaLoadNumber</target>
+        <target>Número de carga de Elisa</target>
         <jms:reference-file line="220">views/RotaVirus/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="a933f7ee47df96debb5cc158abf3667f649647db" resname="rotavirus-form.elisaResult">
         <source>rotavirus-form.elisaResult</source>
-        <target state="new">rotavirus-form.elisaResult</target>
+        <target>Resultado de la prueba de Elisa</target>
         <jms:reference-file line="233">views/RotaVirus/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="e9e7a5e4bb399367df45b6de3be393610f750d01" resname="rotavirus-form.elisaTestDate">
         <source>rotavirus-form.elisaTestDate</source>
-        <target state="new">rotavirus-form.elisaTestDate</target>
+        <target>Fecha de la prueba de Elisa</target>
         <jms:reference-file line="229">views/RotaVirus/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="43691e7306e332f728c2b769b7868644e9734355" resname="rotavirus-form.first-name">
@@ -2669,7 +2670,7 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="f7ab598a8fee0ff1b3a3bbd25c4b881f8d823b3f" resname="rotavirus-form.hasLab">
         <source>rotavirus-form.hasLab</source>
-        <target state="new">rotavirus-form.hasLab</target>
+        <target>Laboratorio vinculado</target>
         <jms:reference-file line="192">views/RotaVirus/show.html.twig</jms:reference-file>
         <jms:reference-file line="251">views/RotaVirus/show.html.twig</jms:reference-file>
       </trans-unit>
@@ -2680,7 +2681,7 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="3391eade9b690848d61a8b32d787176a9cda2ced" resname="rotavirus-form.intensiveCare">
         <source>rotavirus-form.intensiveCare</source>
-        <target state="new">rotavirus-form.intensiveCare</target>
+        <target>Cuidado Intensivo</target>
         <jms:reference-file line="36">Form/Rota/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="bf322c247ab6e7d0b56c21a6fd2730f786dd001a" resname="rotavirus-form.last-name">
@@ -2727,12 +2728,12 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="77425579abb56577a1458f490952f869e0824364" resname="rotavirus-form.siteReceivedDate">
         <source>rotavirus-form.siteReceivedDate</source>
-        <target state="new">rotavirus-form.siteReceivedDate</target>
+        <target>Fecha Recibida</target>
         <jms:reference-file line="196">views/RotaVirus/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="64f3b473dfe49625d94e5f90b3d7bf17fd8df10b" resname="rotavirus-form.state">
         <source>rotavirus-form.state</source>
-        <target state="new">rotavirus-form.state</target>
+        <target>Condición </target>
         <jms:reference-file line="33">Form/Rota/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="c207bb19c6d39e367a58e7a7bd6d78999d37ff2c" resname="rotavirus-form.stoolCollected">
@@ -2752,27 +2753,27 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="71d799cbde3024700bf4b47e764f1932feb6f4bf" resname="rotavirus-form.stoolSentToNL">
         <source>rotavirus-form.stoolSentToNL</source>
-        <target state="new">rotavirus-form.stoolSentToNL</target>
+        <target>Envío al Laboratorio Nacional</target>
         <jms:reference-file line="108">Form/Rota/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="c2dfff1742b6fcc55dff65d629aba1a0e79fc1ad" resname="rotavirus-form.stoolSentToNLDate">
         <source>rotavirus-form.stoolSentToNLDate</source>
-        <target state="new">rotavirus-form.stoolSentToNLDate</target>
+        <target>Fecha que se envió la muestra de heces al Laboratorio Nacional</target>
         <jms:reference-file line="113">Form/Rota/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="4ddb9d195469fc136118b46e50944864b3e7f92e" resname="rotavirus-form.stoolSentToRRL">
         <source>rotavirus-form.stoolSentToRRL</source>
-        <target state="new">rotavirus-form.stoolSentToRRL</target>
+        <target>Enviado al Laboratorio de referencia</target>
         <jms:reference-file line="92">Form/Rota/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="49c8d27176dc1cb18e4f2a69698032428f18a894" resname="rotavirus-form.stoolSentToRRLDate">
         <source>rotavirus-form.stoolSentToRRLDate</source>
-        <target state="new">rotavirus-form.stoolSentToRRLDate</target>
+        <target>Fecha en que se envió la muestra de heces al Laboratorio de Referencia Regional</target>
         <jms:reference-file line="97">Form/Rota/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="3c00b6b0e793684bbb42fd8b24cd01e6ed62592e" resname="rotavirus-form.stored">
         <source>rotavirus-form.stored</source>
-        <target state="new">rotavirus-form.stored</target>
+        <target>Rotavirus guardado</target>
         <jms:reference-file line="204">views/RotaVirus/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="e2433442af01ff86bd9ebdd2ef2fe6500a875ce3" resname="rotavirus-form.symptomDehydration">
@@ -2783,7 +2784,7 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="5df713bb5391b60a8c8d9fcd9d3263be77ea1b30" resname="rotavirus-form.symptomDehydrationAmount">
         <source>rotavirus-form.symptomDehydrationAmount</source>
-        <target>rotavirus-form.symptomDehydrationAmount</target>
+        <target>Cantidad de deshidratación </target>
         <jms:reference-file line="47">Form/Rota/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="387edb7f4d6393d323ba2bcd815a871fec3df82c" resname="rotavirus-form.symptomDiarrhea">
@@ -2856,7 +2857,7 @@ Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="c405fde50a71cd2c0fa822a260d62d55d5e211cf" resname="site-assigned-case-id">
         <source>site-assigned-case-id</source>
-        <target>ID Sitio Sentinela</target>
+        <target>ID Sitio Sentinel</target>
         <jms:reference-file line="38">SentinelBundle/Form/CreateType.php</jms:reference-file>
         <jms:reference-file line="39">SentinelBundle/Form/BaseFilter.php</jms:reference-file>
       </trans-unit>
