diff --git a/app/Resources/translations/messages.es.xliff b/app/Resources/translations/messages.es.xliff
index d1332cc..9802f6c 100644
--- a/app/Resources/translations/messages.es.xliff
+++ b/app/Resources/translations/messages.es.xliff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:jms="urn:jms:translation" version="1.2">
-  <file date="2015-07-01T13:04:49Z" source-language="en" target-language="es" datatype="plaintext" original="not.available">
+  <file date="2016-03-25T10:50:53Z" source-language="en" target-language="es" datatype="plaintext" original="not.available">
     <header>
       <tool tool-id="JMSTranslationBundle" tool-name="JMSTranslationBundle" tool-version="1.1.0-DEV"/>
       <note>The source node in most cases contains the sample message as written by the developer. If it looks like a dot-delimitted string such as "form.label.firstname", then the developer has not provided a default message.</note>
@@ -8,17 +8,17 @@
     <body>
       <trans-unit id="e43d612e11f1568f2373e719d4c4b08dcecdc7cc" resname="Logout">
         <source>Logout</source>
-        <target state="new">Logout</target>
+        <target>Cerrar Sesión</target>
         <jms:reference-file line="289">Resources/views/base.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="cdc22e6692fc5dc99971633a8c13cf512657919e" resname="site-title-text">
         <source>WHO Sentinel Network</source>
-        <target state="new">WHO Sentinel Network</target>
+        <target>Sistema Hospitalario de Vigilancia Centinela para   EPV</target>
         <jms:reference-file line="24">Resources/views/base.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="8aa4d8377d7045bec9b19efae66e98d6b8690acd" resname="sitebrand-text">
         <source>NUVI Sentinel Network</source>
-        <target state="new">NUVI Sentinel Network</target>
+        <target>Sistema Hospitalario de Vigilancia Centinela para   EPV</target>
         <jms:reference-file line="21">Resources/views/base.html.twig</jms:reference-file>
       </trans-unit>
     </body>
diff --git a/src/NS/SentinelBundle/Resources/translations/messages.en.xliff b/src/NS/SentinelBundle/Resources/translations/messages.en.xliff
index 1ec5f0c..c6627dd 100644
--- a/src/NS/SentinelBundle/Resources/translations/messages.en.xliff
+++ b/src/NS/SentinelBundle/Resources/translations/messages.en.xliff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:jms="urn:jms:translation" version="1.2">
-  <file date="2016-03-24T16:29:50Z" source-language="en" target-language="en" datatype="plaintext" original="not.available">
+  <file date="2016-03-25T10:43:58Z" source-language="en" target-language="en" datatype="plaintext" original="not.available">
     <header>
       <tool tool-id="JMSTranslationBundle" tool-name="JMSTranslationBundle" tool-version="1.1.0-DEV"/>
       <note>The source node in most cases contains the sample message as written by the developer. If it looks like a dot-delimitted string such as "form.label.firstname", then the developer has not provided a default message.</note>
@@ -114,7 +114,7 @@
       </trans-unit>
       <trans-unit id="181614cc9373c40e94946fc3af6609d0900b9138" resname="22F/22A">
         <source>22F/22A</source>
-        <target>22F/22A</target>
+        <target></target>
       </trans-unit>
       <trans-unit id="3cb586d73fc8050300afcbd92ec6cfec16d328c6" resname="23A">
         <source>23A</source>
@@ -520,7 +520,7 @@
       </trans-unit>
       <trans-unit id="4f6bb885ab76beef285b275113f6fbbdecb57176" resname="Data Linking">
         <source>Data Linking</source>
-        <target state="new">Data Linking</target>
+        <target>Data Linking</target>
         <jms:reference-file line="2">Report/RotaVirus/data-linking.html.twig</jms:reference-file>
         <jms:reference-file line="2">Report/IBD/data-linking.html.twig</jms:reference-file>
       </trans-unit>
@@ -1286,7 +1286,7 @@
       </trans-unit>
       <trans-unit id="e08b0f0cdec57fe8860ddaf468b9e5654b98cc9d" resname="Results per page">
         <source>Results per page</source>
-        <target state="new">Results per page</target>
+        <target>Results per page</target>
         <jms:reference-file line="21">views/RotaVirus/index.html.twig</jms:reference-file>
         <jms:reference-file line="21">views/IBD/index.html.twig</jms:reference-file>
       </trans-unit>
@@ -1450,7 +1450,7 @@
       </trans-unit>
       <trans-unit id="8cd9373697bbf06a7131190c7884bfbefb226aa9" resname="Symptoms">
         <source>Symptoms</source>
-        <target state="new">Symptoms</target>
+        <target>Symptoms</target>
         <jms:reference-file line="15">views/RotaVirus/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="e13672b32cc497b95ef94e1ab13419a0f9f22362" resname="Symptoms/signs related to meningitis">
@@ -1839,12 +1839,14 @@
       </trans-unit>
       <trans-unit id="0263dfd0afae8ee3b162f51e20acde9abb221e9b" resname="ibd-form.blood-sent-to-rrl-date">
         <source>ibd-form.blood-sent-to-rrl-date</source>
-        <target state="new">ibd-form.blood-sent-to-rrl-date</target>
+        <target>Date when isolate from blood was sent to RL
+</target>
         <jms:reference-file line="204">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="47273c1794b96aa871e2d953555f1e69235cd108" resname="ibd-form.broth-sent-to-rrl-date">
         <source>ibd-form.broth-sent-to-rrl-date</source>
-        <target state="new">ibd-form.broth-sent-to-rrl-date</target>
+        <target>Date when blood broth was sent to RL
+</target>
         <jms:reference-file line="205">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="e5ba7a020342f19b8f355132f7dd2674f76c52e0" resname="ibd-form.case-id">
@@ -1900,7 +1902,7 @@
       </trans-unit>
       <trans-unit id="592e81911f3db51a32610e8f0ba4421adc702f03" resname="ibd-form.csf-culture-contaminant">
         <source>ibd-form.csf-culture-contaminant</source>
-        <target state="new">ibd-form.csf-culture-contaminant</target>
+        <target>ibd-form.csf-culture-contaminant</target>
         <jms:reference-file line="60">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="3332d66e09ea8b6dd4eca625c491c87904e6aa2a" resname="ibd-form.csf-culture-other">
@@ -1943,7 +1945,7 @@
       </trans-unit>
       <trans-unit id="75fb3d80d4a8e37fb9daf249e9197bed37ba6c96" resname="ibd-form.csf-isol-sent-to-rrl-date">
         <source>ibd-form.csf-isol-sent-to-rrl-date</source>
-        <target state="new">ibd-form.csf-isol-sent-to-rrl-date</target>
+        <target>Date when isolate from CSF was sent to RL</target>
         <jms:reference-file line="203">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="5bd6a0e287156c72e1ca281ce63053256252457c" resname="ibd-form.csf-lab-datetime">
@@ -1991,7 +1993,7 @@
       </trans-unit>
       <trans-unit id="917bd0ef3ef2d3df7e4c6185d959d676c56a5241" resname="ibd-form.csf-sent-to-rrl-date">
         <source>ibd-form.csf-sent-to-rrl-date</source>
-        <target state="new">ibd-form.csf-sent-to-rrl-date</target>
+        <target>Date when CSF sample was sent to RL</target>
         <jms:reference-file line="202">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="6f3302e375553355129bf6ddd6e5e3ac374d6783" resname="ibd-form.csf-store">
@@ -2267,7 +2269,7 @@
       </trans-unit>
       <trans-unit id="dac15dc0e75761c8518c20cfd8b4c291e4ae66b9" resname="ibd-form.other-test-result">
         <source>ibd-form.other-test-result</source>
-        <target state="new">ibd-form.other-test-result</target>
+        <target>Results of other test performed in the lab</target>
         <jms:reference-file line="170">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="200fbe0f4392a0194affd6dada0367aa49ad8166" resname="ibd-form.otherSpecimenCollected">
@@ -2609,7 +2611,7 @@
       </trans-unit>
       <trans-unit id="afc949b4dd71f9655e84e5ee9bbec57836fc2eb3" resname="ibd.spnSerotype">
         <source>ibd.spnSerotype</source>
-        <target>spn Serotype</target>
+        <target>Spn Serotype</target>
         <jms:reference-file line="293">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="327">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
diff --git a/src/NS/SentinelBundle/Resources/translations/messages.es.xliff b/src/NS/SentinelBundle/Resources/translations/messages.es.xliff
index 20128e0..5bd063d 100644
--- a/src/NS/SentinelBundle/Resources/translations/messages.es.xliff
+++ b/src/NS/SentinelBundle/Resources/translations/messages.es.xliff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:jms="urn:jms:translation" version="1.2">
-  <file date="2015-07-01T13:05:00Z" source-language="en" target-language="es" datatype="plaintext" original="not.available">
+  <file date="2016-03-25T18:34:46Z" source-language="en" target-language="es" datatype="plaintext" original="not.available">
     <header>
       <tool tool-id="JMSTranslationBundle" tool-name="JMSTranslationBundle" tool-version="1.1.0-DEV"/>
       <note>The source node in most cases contains the sample message as written by the developer. If it looks like a dot-delimitted string such as "form.label.firstname", then the developer has not provided a default message.</note>
@@ -8,17 +8,17 @@
     <body>
       <trans-unit id="da39a3ee5e6b4b0d3255bfef95601890afd80709" resname="">
         <source></source>
-        <target state="new"></target>
+        <target></target>
         <jms:reference-file line="31">Form/Types/CTValueType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="18ab4daacc3850d1f77d27057bb1ec6d69b46aa7" resname="%count% Results">
         <source>%count% Results</source>
-        <target state="new">%count% Results</target>
+        <target>Resultados: %count% </target>
         <jms:reference-file line="9">views/Report/fieldPopulation.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="c04cd8a70157ece1018ac829accacdeaeb9679c9" resname="0-5">
         <source>0-5</source>
-        <target state="new">0-5</target>
+        <target>0-5</target>
         <jms:reference-file line="17">views/Report/annualAgeDistribution.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="308ef13db5871494c2383c8cf00970521e3e9ef9" resname="1 dose">
@@ -39,7 +39,7 @@
       </trans-unit>
       <trans-unit id="b0b18ba89cd0608bdc21b8003cec554abbb180b9" resname="12-23">
         <source>12-23</source>
-        <target state="new">12-23</target>
+        <target>12-23</target>
         <jms:reference-file line="19">views/Report/annualAgeDistribution.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="46a548c4f84f29214f382b268fbab0dfc6f72ca2" resname="12A">
@@ -120,7 +120,7 @@
       </trans-unit>
       <trans-unit id="5c19a841c96550e3a98764221d911dbdbe0cf386" resname="24-59">
         <source>24-59</source>
-        <target state="new">24-59</target>
+        <target>24-59</target>
         <jms:reference-file line="20">views/Report/annualAgeDistribution.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="f6d456ab4c20b10f4f9dcdae6a2e3743aa4f9de3" resname="24A/24B/24F">
@@ -157,7 +157,7 @@
       </trans-unit>
       <trans-unit id="ff7537e61ade4814f5afaa0d943fdf431e62c7d7" resname="6-11">
         <source>6-11</source>
-        <target state="new">6-11</target>
+        <target>6-11</target>
         <jms:reference-file line="18">views/Report/annualAgeDistribution.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="be638c6b064202bcb509c3184d55ff400215f2ed" resname="6A">
@@ -279,7 +279,7 @@
       </trans-unit>
       <trans-unit id="42b657c8bebf02e57f682d03428fbdc6bcae2837" resname="Blood culture broth">
         <source>Blood culture broth</source>
-        <target state="new">Blood culture broth</target>
+        <target>Blood culture broth</target>
       </trans-unit>
       <trans-unit id="0504fc68c397e6f1d5a2d8342b06e545a890d772" resname="Bloody">
         <source>Bloody</source>
@@ -287,11 +287,12 @@
       </trans-unit>
       <trans-unit id="c70830a91bab7a8b117d115cb089d45a9663de74" resname="Both (ORS/ORT and IV fluids)">
         <source>Both (ORS/ORT and IV fluids)</source>
-        <target state="new">Both (ORS/ORT and IV fluids)</target>
+        <target>Ambos (
+Both (ORS/ORT and IV fluids)</target>
       </trans-unit>
       <trans-unit id="d5b8ecfb0f8c2745aa1d63559a0e837f57b8df70" resname="By Country">
         <source>By Country</source>
-        <target state="new">By Country</target>
+        <target>By Country</target>
         <jms:reference-file line="48">views/Security/homepage.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="32aed76fc277287cf218e00c2786cfeee22c813f" resname="By Diagnosis">
@@ -302,7 +303,7 @@
       </trans-unit>
       <trans-unit id="32e1bbe2f5c8a9c4483c352b659be1e0f7d9c46a" resname="By Site">
         <source>By Site</source>
-        <target state="new">By Site</target>
+        <target>Por Sitio</target>
         <jms:reference-file line="70">views/Security/homepage.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="32096c2e0eff33d844ee6d675407ace18289357d" resname="C">
@@ -316,11 +317,11 @@
       </trans-unit>
       <trans-unit id="075a704274c71eb4f91f3b8d6e34f3fea19765f9" resname="CSF + Isolate">
         <source>CSF + Isolate</source>
-        <target state="new">CSF + Isolate</target>
+        <target>CSF + Aislar</target>
       </trans-unit>
       <trans-unit id="775275cfc23d95b3976ef3bbea43f42a663ffdda" resname="CT Values From PCR">
         <source>CT Values From PCR</source>
-        <target state="new">CT Values From PCR</target>
+        <target>CT Valores de PCR</target>
         <jms:reference-file line="21">views/IBD/editBaseLab.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="77dfd2135f4db726c47299bb55be26f7f4525a46" resname="Cancel">
@@ -361,7 +362,7 @@
       </trans-unit>
       <trans-unit id="589acc5d25d7be341d54af342e812f3cc85865e6" resname="Case Progression">
         <source>Case Progression</source>
-        <target state="new">Case Progression</target>
+        <target>Progreso del caso</target>
         <jms:reference-file line="102">views/Report/annualAgeDistribution.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="719ea396ad92e01b4757ec2b93bb1e5f270f771d" resname="Clear">
@@ -378,12 +379,12 @@
       </trans-unit>
       <trans-unit id="adac69379a626c2436948a4ef1792c7d719ef929" resname="Code">
         <source>Code</source>
-        <target state="new">Code</target>
+        <target>Código</target>
         <jms:reference-file line="54">SentinelBundle/Admin/ReferenceLabAdmin.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="22c36db366e95f54cde8dc00859b31205a7c19b8" resname="Collection Info">
         <source>Collection Info</source>
-        <target state="new">Collection Info</target>
+        <target>Colección de información </target>
         <jms:reference-file line="18">views/IBD/editLab.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="49252e62e00fc748d88e9116e4f339147369a405" resname="Common Reports">
@@ -427,11 +428,11 @@
       </trans-unit>
       <trans-unit id="1282bad209e4796bde39661d95a44c1be496b25c" resname="Country API">
         <source>Country API</source>
-        <target state="new">Country API</target>
+        <target>País API</target>
       </trans-unit>
       <trans-unit id="2503248972e872a7a90502577f453c4393a63475" resname="Country Import/Export">
         <source>Country Import/Export</source>
-        <target state="new">Country Import/Export</target>
+        <target>País Importar/Exportar</target>
       </trans-unit>
       <trans-unit id="24d5b7573e0f17f42038919c6ee16b45f5c248e6" resname="Country Statistics">
         <source>Country Statistics</source>
@@ -441,7 +442,7 @@
       </trans-unit>
       <trans-unit id="1add43318b227e91ce6d3aae7ff4b4b31f45a0b0" resname="Create Case">
         <source>Create Case</source>
-        <target state="new">Create Case</target>
+        <target>Crear caso</target>
         <jms:reference-file line="54">views/RotaVirus/index.html.twig</jms:reference-file>
         <jms:reference-file line="55">views/IBD/index.html.twig</jms:reference-file>
       </trans-unit>
@@ -467,7 +468,7 @@
       </trans-unit>
       <trans-unit id="daca76f341170488ecbef5e895d310588588cea6" resname="Csf spn pcr positive">
         <source>Csf spn pcr positive</source>
-        <target state="new">Csf spn pcr positive</target>
+        <target>Csf spn pcr positivo</target>
         <jms:reference-file line="26">views/Report/fieldPopulation.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="71f440a1dd4cf93fe583dd09db3c514d1b6820c4" resname="Culture Positive and Additional Lab Confirmed Cases By Diagnostic Test">
@@ -478,17 +479,17 @@
       </trans-unit>
       <trans-unit id="63b2725c3a0e2eb9eae36bb47c96a8b55fec2e75" resname="Culture+">
         <source>Culture+</source>
-        <target state="new">Culture+</target>
+        <target>Cultivo+</target>
         <jms:reference-file line="17">views/Report/culturePositive.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="02b446c95d397c4cee18df3da9854e03ed7e89f4" resname="Culture- &amp; Binax+">
         <source><![CDATA[Culture- & Binax+]]></source>
-        <target state="new"><![CDATA[Culture- & Binax+]]></target>
+        <target><![CDATA[Cultivo- & Binax+]]></target>
         <jms:reference-file line="18">views/Report/culturePositive.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="2436d4d457b8bd6c456fd231fb6211c8ab48bb93" resname="DNA Extract">
         <source>DNA Extract</source>
-        <target state="new">DNA Extract</target>
+        <target>Extracto de ADN</target>
       </trans-unit>
       <trans-unit id="d87f47b47e4d5794d53fe0734c49417bd046cde0" resname="Dashboard">
         <source>Dashboard</source>
@@ -522,7 +523,8 @@
       </trans-unit>
       <trans-unit id="9e9cf3221a30246219863f1d2366e36cb580debc" resname="Documentation">
         <source>Documentation</source>
-        <target state="new">Documentation</target>
+        <target>Documentación
+</target>
         <jms:reference-file line="54">SentinelBundle/Menu/Builder.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="abb454f7083b889c5d49dedd8fca52d3e3a35e35" resname="EPI">
@@ -536,7 +538,7 @@
       </trans-unit>
       <trans-unit id="8fc5633aaf9deadef041285a13d1932225e44e7a" resname="Edit %type% For Case %id%">
         <source>Edit %type% For Case %id%</source>
-        <target state="new">Edit %type% For Case %id%</target>
+        <target>Editar %type% por cada caso %id%</target>
         <jms:reference-file line="4">views/RotaVirus/editBaseLab.html.twig</jms:reference-file>
         <jms:reference-file line="4">views/IBD/editBaseLab.html.twig</jms:reference-file>
       </trans-unit>
@@ -575,7 +577,8 @@
       </trans-unit>
       <trans-unit id="183f6c9b5b650d03aa4ae7c8d7c6ba7b81567727" resname="Field Population">
         <source>Field Population</source>
-        <target state="new">Field Population</target>
+        <target>Población de área 
+</target>
         <jms:reference-file line="2">views/Report/fieldPopulation.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="96e578211aa295317cf257310712fa28ccd8f6c6" resname="Filters">
@@ -654,17 +657,18 @@
       </trans-unit>
       <trans-unit id="efff659b95e53ca68c3fe9a672c17a72183b3d9e" resname="Genotype">
         <source>Genotype</source>
-        <target state="new">Genotype</target>
+        <target>Genotipo
+</target>
         <jms:reference-file line="18">views/RotaVirus/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="83b5ccff29458393b2e256062ff4a7594f40fa76" resname="Genotyping">
         <source>Genotyping</source>
-        <target state="new">Genotyping</target>
+        <target>Proceso de genotipo</target>
         <jms:reference-file line="19">views/RotaVirus/editLab.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="f3dd4543ce161fe06cf1ba8fcc0bff7d6b1370a8" resname="Genotyping Results">
         <source>Genotyping Results</source>
-        <target state="new">Genotyping Results</target>
+        <target>Resultados de genotipos</target>
         <jms:reference-file line="20">views/RotaVirus/editBaseLab.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="4ff0ff91948f32f76ba7932b7290ac166d1cc17c" resname="Gm neg coccobacilli">
@@ -701,7 +705,7 @@
       </trans-unit>
       <trans-unit id="da71976cb422306d363e7e69ce09adb388ce3dda" resname="Hi Result">
         <source>Hi Result</source>
-        <target state="new">Hi Result</target>
+        <target>Resultados Hi</target>
         <jms:reference-file line="29">views/Report/fieldPopulation.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="44dd14f03d1c5a08fdcb240040c5d1ecea86ae23" resname="I forgot my password">
@@ -711,7 +715,7 @@
       </trans-unit>
       <trans-unit id="2a71f5e790a5bd16a54986e1eb5e7a3a69846528" resname="IV fluids">
         <source>IV fluids</source>
-        <target state="new">IV fluids</target>
+        <target>Fluidos Intravenoso</target>
       </trans-unit>
       <trans-unit id="af4d4b42d93ccbd5d03b1cb5906ae3d47221bd45" resname="Incomplete investigation">
         <source>Incomplete investigation</source>
@@ -727,11 +731,11 @@
       </trans-unit>
       <trans-unit id="6c4f80256bc149592c7d0fe3f17522d7c618e156" resname="Inoculated">
         <source>Inoculated</source>
-        <target state="new">Inoculated</target>
+        <target>Inoculado</target>
       </trans-unit>
       <trans-unit id="ce74028d54f7533ed149c52db2e9b3c5fcfffc3e" resname="Intense Support">
         <source>Intense Support</source>
-        <target state="new">Intense Support</target>
+        <target>Apoyo Intenso</target>
         <jms:reference-file line="17">views/Report/fieldPopulation.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="54c2bc11f746b30e16b38457e62f445d4965c330" resname="Interstitial infiltrate">
@@ -745,7 +749,7 @@
       </trans-unit>
       <trans-unit id="28d67e9b9c3e281b3f42497b290ad4a803463042" resname="Isolate">
         <source>Isolate</source>
-        <target state="new">Isolate</target>
+        <target>Aislado</target>
       </trans-unit>
       <trans-unit id="6475463840e57f1dc21360a72f14d2a1c25b2406" resname="LAB">
         <source>LAB</source>
@@ -764,28 +768,28 @@
       </trans-unit>
       <trans-unit id="db0932c1b52936ad2d450f3e3635ae747d95c560" resname="Lab Blood">
         <source>Lab Blood</source>
-        <target state="new">Lab Blood</target>
+        <target>Laboratorio de sangre</target>
         <jms:reference-file line="23">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="2a8ec75786f2176977495776313819e8075bd734" resname="Lab CSF">
         <source>Lab CSF</source>
-        <target state="new">Lab CSF</target>
+        <target>Laboratorio  CSF</target>
         <jms:reference-file line="22">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="d50bc05be0e9b0f49860413c3121d65e4112d904" resname="Lab Information">
         <source>Lab Information</source>
-        <target state="new">Lab Information</target>
+        <target>Laboratorio de Información </target>
         <jms:reference-file line="19">views/RotaVirus/editBaseLab.html.twig</jms:reference-file>
         <jms:reference-file line="18">views/IBD/editBaseLab.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="18f17c05e2c86b15dc0c6b17fe2b8d184faaa33e" resname="Lab Other Fluid">
         <source>Lab Other Fluid</source>
-        <target state="new">Lab Other Fluid</target>
+        <target>Laboratorio de otros fluidos</target>
         <jms:reference-file line="24">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="e3f535c7d82c4f97e1e3d09c16cfa3015b88d24b" resname="Lab Serotype">
         <source>Lab Serotype</source>
-        <target state="new">Lab Serotype</target>
+        <target>Laboratorio de serotipo</target>
         <jms:reference-file line="25">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="955eabd1c90a1eb745aabc1978cd51b11990901d" resname="Lab-confirmed for HI">
@@ -816,7 +820,7 @@
       </trans-unit>
       <trans-unit id="43e939b5fb23c589bd8309303803ca4044b1b7a5" resname="Latest IBD Cases">
         <source>Latest IBD Cases</source>
-        <target state="new">Latest IBD Cases</target>
+        <target>Últimos casos de IBD </target>
         <jms:reference-file line="10">views/IBD/index.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="dfb450c7e95da7c8d07b85a3f8e61d398ff68681" resname="Latest Rotavirus Cases">
@@ -830,7 +834,8 @@
       </trans-unit>
       <trans-unit id="61d14bfd8db9302c47a91363bfc3c215f747d391" resname="Lists the last %count% IBD cases">
         <source>Lists the last %count% IBD cases</source>
-        <target state="new">Lists the last %count% IBD cases</target>
+        <target>Lista de últimos casos de IBD %count% 
+</target>
         <jms:reference-file line="3">views/IBD/index.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="6f604eed838cef726bbc73efd2a61ab5d6b7bdd4" resname="Lists the last %count% rotavirus cases">
@@ -859,7 +864,8 @@
       </trans-unit>
       <trans-unit id="7e4ac6803c9159c694f63d089cb06b2519c16aba" resname="Methods">
         <source>Methods</source>
-        <target state="new">Methods</target>
+        <target>Métodos
+</target>
         <jms:reference-file line="20">views/IBD/editBaseLab.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="543c11922fafa6bc94f9277ba87081eb9ac9f1dd" resname="Mixed (specify)">
@@ -887,7 +893,7 @@
       </trans-unit>
       <trans-unit id="727d00eb2ea7df33a9b7f2a6ab4bb65ec77db833" resname="National Lab">
         <source>National Lab</source>
-        <target state="new">National Lab</target>
+        <target>Laboratorio Nacional</target>
         <jms:reference-file line="30">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="c70827c69d7f2e385869c21a9756535533dbe203" resname="Negative">
@@ -924,7 +930,7 @@
       </trans-unit>
       <trans-unit id="9e8a62af45b5c99758897be76938b1c20b38319e" resname="Non-result choices">
         <source>Non-result choices</source>
-        <target state="new">Non-result choices</target>
+        <target>Opciones sin resultados</target>
         <jms:reference-file line="30">Form/Types/CTValueType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="c565a07d6e668148b9369efd7252f062b1f572f2" resname="Non-type b">
@@ -961,17 +967,18 @@
       </trans-unit>
       <trans-unit id="94ca3343bc35b6633fc5f605ae8e057de98a148f" resname="Number Enrolled By Month &amp; Suspected Diagnosis">
         <source><![CDATA[Number Enrolled By Month & Suspected Diagnosis]]></source>
-        <target state="new"><![CDATA[Number Enrolled By Month & Suspected Diagnosis]]></target>
+        <target>Número de inscritos por mes y por diagnósticos sospechosos</target>
         <jms:reference-file line="79">views/Report/annualAgeDistribution.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="e8c8fb09a83d86403479132c086fdf365be367fb" resname="Number and Percent Enrolled: Admit Diagnosis">
         <source>Number and Percent Enrolled: Admit Diagnosis</source>
-        <target state="new">Number and Percent Enrolled: Admit Diagnosis</target>
+        <target>Número y porcentaje de inscritos: 
+Administrar diagnosis </target>
         <jms:reference-file line="2">views/Report/percentEnrolled.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="4c100b84b4ee887ba44105eb25d52e008800b694" resname="ORS/ORT and/or IV fluids and/or Other/Multiple">
         <source>ORS/ORT and/or IV fluids and/or Other/Multiple</source>
-        <target state="new">ORS/ORT and/or IV fluids and/or Other/Multiple</target>
+        <target>ORS/ORT y/o fluidos IV y/u Otro/Múltiple</target>
       </trans-unit>
       <trans-unit id="cf9b77061f7b3126b49d50a6fa68f7ca8c26b7a3" resname="Open">
         <source>Open</source>
@@ -979,7 +986,7 @@
       </trans-unit>
       <trans-unit id="2975e8d694a2d53205e4042a7a138b9bd38df9ad" resname="Oral - ORS/ORT">
         <source>Oral - ORS/ORT</source>
-        <target state="new">Oral - ORS/ORT</target>
+        <target>Oral - ORS/ORT</target>
       </trans-unit>
       <trans-unit id="6e6a6f2086bb5fe5dbfd17d8d5f502d48759834b" resname="Other">
         <source>Other</source>
@@ -1012,7 +1019,7 @@
       </trans-unit>
       <trans-unit id="799618173f61bf5dc4b1ca23bbebeb2a226b4e68" resname="PCR+ &amp; Binax- &amp; Culture-">
         <source><![CDATA[PCR+ & Binax- & Culture-]]></source>
-        <target state="new"><![CDATA[PCR+ & Binax- & Culture-]]></target>
+        <target><![CDATA[PCR+ & Binax- & Cultivo-]]></target>
         <jms:reference-file line="19">views/Report/culturePositive.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="6d763194d2654385226ca98f9757372bb32e49d5" resname="PCV10">
@@ -1101,7 +1108,7 @@
       </trans-unit>
       <trans-unit id="e3b21a540a718b7284f94d7105bb256a826d03ca" resname="Please Provide Your Email Address">
         <source>Please Provide Your Email Address</source>
-        <target state="new">Please Provide Your Email Address</target>
+        <target>Por favor provee su correo electrónico </target>
         <jms:reference-file line="25">views/Security/forgotPassword.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="a7942f226ed1080246f2b2fde35b11e19a91e6a8" resname="Please Select...">
@@ -1114,7 +1121,7 @@
       </trans-unit>
       <trans-unit id="47658dced1ec37586a51bf23d66051c62ff85998" resname="Pleural Fluid">
         <source>Pleural Fluid</source>
-        <target state="new">Pleural Fluid</target>
+        <target>Fluido Pleural</target>
       </trans-unit>
       <trans-unit id="baaf165357b4ddb22a1db71334044792390ac952" resname="Pleural effusion">
         <source>Pleural effusion</source>
@@ -1165,7 +1172,7 @@
       </trans-unit>
       <trans-unit id="e53938a82e4c619e6efb6f1f7b65ff2d99368bd2" resname="Reference Lab">
         <source>Reference Lab</source>
-        <target state="new">Reference Lab</target>
+        <target>Laboratorio de referencia </target>
         <jms:reference-file line="31">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="0f217179940c6d89f5cb2c7002a58d91ab7286c1" resname="Region">
@@ -1179,7 +1186,7 @@
       </trans-unit>
       <trans-unit id="b846d1c08e6f20ab3876a6108664a6e90d0e469d" resname="Region Import/Export">
         <source>Region Import/Export</source>
-        <target state="new">Region Import/Export</target>
+        <target>Region Importar/Exportar</target>
       </trans-unit>
       <trans-unit id="bab9bc458f38d4d574e5baa63c72341db6ddaa30" resname="Repeat Password">
         <source>Repeat Password</source>
@@ -1189,19 +1196,19 @@
       </trans-unit>
       <trans-unit id="3fb75e3bfe4de94eb5198656fa9de95352dab915" resname="Reset Password">
         <source>Reset Password</source>
-        <target state="new">Reset Password</target>
+        <target>Reiniciar contraseña </target>
         <jms:reference-file line="40">views/Security/forgotPassword.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="612e12d29278b5519294bc25cdaddffec6d0f1c6" resname="Results">
         <source>Results</source>
-        <target state="new">Results</target>
+        <target>Resultados </target>
         <jms:reference-file line="9">views/Report/percentEnrolled.html.twig</jms:reference-file>
         <jms:reference-file line="10">views/Report/culturePositive.html.twig</jms:reference-file>
         <jms:reference-file line="10">views/Report/annualAgeDistribution.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="793fee1e4269a5b4bf830a2daf4c5b8f5e45eb67" resname="RotaTeq, Merck">
         <source>RotaTeq, Merck</source>
-        <target state="new">RotaTeq, Merck</target>
+        <target>RotaTeq, Merck</target>
       </trans-unit>
       <trans-unit id="b6a30e040e2c8bc16a3b2e95e725155b4b3fdce5" resname="Rotarix, GSK">
         <source>Rotarix, GSK</source>
@@ -1209,12 +1216,12 @@
       </trans-unit>
       <trans-unit id="19e30cbb1fe4142ac663856b78b6b9efcadb3936" resname="Sample &amp; Elisa">
         <source><![CDATA[Sample & Elisa]]></source>
-        <target state="new"><![CDATA[Sample & Elisa]]></target>
+        <target><![CDATA[Muestra & Elisa]]></target>
         <jms:reference-file line="18">views/RotaVirus/editLab.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="cf3d11ff8a7b7eeb1a58d8bbfbbaf8311406e122" resname="Sample Information">
         <source>Sample Information</source>
-        <target state="new">Sample Information</target>
+        <target>Muestra de información </target>
         <jms:reference-file line="19">views/IBD/editBaseLab.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="efc007a393f66cdb14d57d385822a3d9e36ef873" resname="Save">
@@ -1232,16 +1239,16 @@
       </trans-unit>
       <trans-unit id="ba415c6ec1533043b9d3949af02c64156fb34f37" resname="Save &amp; Close">
         <source><![CDATA[Save & Close]]></source>
-        <target state="new"><![CDATA[Save & Close]]></target>
+        <target>Salvar y Cerrar</target>
         <jms:reference-file line="144">views/IBD/edit.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="186238cccd6fdd63822aa5886b1950d4f8c61dc5" resname="Sepsis">
         <source>Sepsis</source>
-        <target state="new">Sepsis</target>
+        <target>Septicemia</target>
       </trans-unit>
       <trans-unit id="24cee23fc38f509d1ed91d4b08e5d052fefe2ea2" resname="Serotype / Serogroup Results">
         <source>Serotype / Serogroup Results</source>
-        <target state="new">Serotype / Serogroup Results</target>
+        <target>Resultados de Serotipo / Serogrupo </target>
         <jms:reference-file line="22">views/IBD/editBaseLab.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="137b4f2ee499ee6f9aadb03f117df663a69aabd3" resname="Serotype Results">
@@ -1262,7 +1269,7 @@
       </trans-unit>
       <trans-unit id="799b6553db51482380a99f5129803abdccded031" resname="Site API">
         <source>Site API</source>
-        <target state="new">Site API</target>
+        <target>Sitio API</target>
       </trans-unit>
       <trans-unit id="baaabfc71e818fe50aefacb38b9f9a6beb219123" resname="Site Case ID">
         <source>Site Case ID</source>
@@ -1273,11 +1280,11 @@
       </trans-unit>
       <trans-unit id="501fbc593e79084b78bc093c5b3331e7303a780c" resname="Site Import/Export">
         <source>Site Import/Export</source>
-        <target state="new">Site Import/Export</target>
+        <target>Sitio de Importar/Exportar </target>
       </trans-unit>
       <trans-unit id="f70c03081c5209cc4aa55b21d6f3911aa88a33e0" resname="Site Lab">
         <source>Site Lab</source>
-        <target state="new">Site Lab</target>
+        <target>Laboratorio Local</target>
       </trans-unit>
       <trans-unit id="00b2aad45d07b70e1b7eefea74316913901b6e42" resname="Some">
         <source>Some</source>
@@ -1301,7 +1308,7 @@
       </trans-unit>
       <trans-unit id="e306248f0d41edc7373128a8677e69a684fe7ad8" resname="Spn Result">
         <source>Spn Result</source>
-        <target state="new">Spn Result</target>
+        <target>Resultado de Spn</target>
         <jms:reference-file line="28">views/Report/fieldPopulation.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="bae7d5be70820ed56467bd9a63744e23b47bd711" resname="Status">
@@ -1313,7 +1320,7 @@
       </trans-unit>
       <trans-unit id="ab646fbf4358000ca0b02c28ddb3bf3c76191963" resname="Suspect">
         <source>Suspect</source>
-        <target state="new">Suspect</target>
+        <target>Sospecha Clínica </target>
       </trans-unit>
       <trans-unit id="e2cbe6c8e35a0c06ef807e763165a5ca499b6237" resname="Suspected">
         <source>Suspected</source>
@@ -1329,20 +1336,20 @@
       </trans-unit>
       <trans-unit id="7b18d85ccaae50c2f88c2e6b9222d29547ece832" resname="Suspected sepsis">
         <source>Suspected sepsis</source>
-        <target state="new">Suspected sepsis</target>
+        <target>Sospecha clínica de Septicemia</target>
       </trans-unit>
       <trans-unit id="3c4e7e374b8f093a9175523a9788473c2904dfdb" resname="Suspected severe pneumonia">
         <source>Suspected severe pneumonia</source>
-        <target state="new">Suspected severe pneumonia</target>
+        <target>Sospecha de neumonía severa</target>
       </trans-unit>
       <trans-unit id="e13672b32cc497b95ef94e1ab13419a0f9f22362" resname="Symptoms/signs related to meningitis">
         <source>Symptoms/signs related to meningitis</source>
-        <target state="new">Symptoms/signs related to meningitis</target>
+        <target>Síntomas/señales relacionadas con la meningitis </target>
         <jms:reference-file line="100">views/IBD/edit.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="f530beb14362ec2a13b2986956ae0c14a24002e7" resname="Symptoms/signs related to pneumonia">
         <source>Symptoms/signs related to pneumonia</source>
-        <target state="new">Symptoms/signs related to pneumonia</target>
+        <target>Síntomas/señales relacionadas con la neumonía </target>
         <jms:reference-file line="118">views/IBD/edit.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="831377cf9256094e86e41449c9a0b0d1160af443" resname="System Id">
@@ -1358,12 +1365,12 @@
       </trans-unit>
       <trans-unit id="b25928c69902557b0ef0a628490a3a1768d7b82f" resname="Total">
         <source>Total</source>
-        <target state="new">Total</target>
+        <target>Total</target>
         <jms:reference-file line="95">views/Report/annualAgeDistribution.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="1208423354693a4ecea29f76f0627bcf0eaa080d" resname="Total # of Cases">
         <source>Total # of Cases</source>
-        <target state="new">Total # of Cases</target>
+        <target>Total # de casos</target>
         <jms:reference-file line="18">views/Report/fieldPopulation.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="f76d1e2a50543a098caaec8da67e10271c0a83af" resname="Transferred">
@@ -1399,7 +1406,7 @@
       </trans-unit>
       <trans-unit id="0c8186da01f87243ac37efa00fe66b84d4e0ce88" resname="Whole Blood">
         <source>Whole Blood</source>
-        <target state="new">Whole Blood</target>
+        <target>Sangre Pura</target>
       </trans-unit>
       <trans-unit id="c032adc1ff629c9b66f22749ad667e6beadf144b" resname="X">
         <source>X</source>
@@ -1419,13 +1426,13 @@
       </trans-unit>
       <trans-unit id="879e32326c5217968ac8e3a14e4a8a5214957264" resname="Year">
         <source>Year</source>
-        <target state="new">Year</target>
+        <target>Año</target>
         <jms:reference-file line="16">views/Report/culturePositive.html.twig</jms:reference-file>
         <jms:reference-file line="95">views/Report/annualAgeDistribution.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="28ca72dbc84242332b42d831148af36cd198fbf7" resname="Year &amp; Total">
         <source><![CDATA[Year & Total]]></source>
-        <target state="new"><![CDATA[Year & Total]]></target>
+        <target>Año y Total</target>
         <jms:reference-file line="16">views/Report/annualAgeDistribution.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="5397e0583f14f6c88de06b1ef28f460a1fb5b0ae" resname="Yes">
@@ -1449,7 +1456,7 @@
       </trans-unit>
       <trans-unit id="6e36913c5daaf91b67d2edb4a74f4c7bf70ffca7" resname="[Enter value]">
         <source>[Enter value]</source>
-        <target state="new">[Enter value]</target>
+        <target>Introduzca Valor</target>
         <jms:reference-file line="30">Form/Types/CTValueType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="86f7e437faa5a7fce15d1ddcb9eaeaea377667b8" resname="a">
@@ -1463,12 +1470,12 @@
       </trans-unit>
       <trans-unit id="bd979f0607061e62ae45e30554a9d1ff1c2754d3" resname="admin.form-can-create-national-lab-record">
         <source>admin.form-can-create-national-lab-record</source>
-        <target state="new">admin.form-can-create-national-lab-record</target>
+        <target>Puede Crearse un Laboratorio Nacional de Resultados</target>
         <jms:reference-file line="99">SentinelBundle/Admin/UserAdmin.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="6ce41a81103332954778b43d4e624fcc694f6c69" resname="admin.form-can-create-reference-lab-record">
         <source>admin.form-can-create-reference-lab-record</source>
-        <target state="new">admin.form-can-create-reference-lab-record</target>
+        <target>Puede Crearse un Laboratorio de Can Create Reference Lab Resultsadmin.form-can-create-reference-lab-record</target>
         <jms:reference-file line="98">SentinelBundle/Admin/UserAdmin.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="83dc047b895b5a3ea8064c71b7bea957ee38c40e" resname="admin.form-can-create-sitelab-record">
@@ -1478,7 +1485,7 @@
       </trans-unit>
       <trans-unit id="f45037b362769913c647f9bd5f755b01f22d2120" resname="admin.form-reference-lab">
         <source>admin.form-reference-lab</source>
-        <target state="new">admin.form-reference-lab</target>
+        <target>admin.forma-laboratorio-referencia</target>
         <jms:reference-file line="100">SentinelBundle/Admin/UserAdmin.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="e9d71f5ee7c92d6dc9e92ffdad17b8bd49418f98" resname="b">
@@ -1487,17 +1494,17 @@
       </trans-unit>
       <trans-unit id="49e1d7a45d0316b95e2388d2dc2b0f1b94a33dbc" resname="blood Collected">
         <source>blood Collected</source>
-        <target state="new">blood Collected</target>
+        <target>Sangre Colectada </target>
         <jms:reference-file line="21">views/Report/fieldPopulation.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="cfcd821ea6e7348be0b6c0de8ed4a4383c1ae01e" resname="blood Equal">
         <source>blood Equal</source>
-        <target state="new">blood Equal</target>
+        <target></target>
         <jms:reference-file line="23">views/Report/fieldPopulation.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="a53658753e1d351efc751c39527cd0994c80bf81" resname="blood Result">
         <source>blood Result</source>
-        <target state="new">blood Result</target>
+        <target>Resultados de Sangre</target>
         <jms:reference-file line="22">views/Report/fieldPopulation.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="84a516841ba77a5b4648de2cd0dfcb30ea46dbb4" resname="c">
@@ -1506,22 +1513,22 @@
       </trans-unit>
       <trans-unit id="e23e32ba593b3f97acca179a8075c1aab17ecf53" resname="csf Binax Result">
         <source>csf Binax Result</source>
-        <target state="new">csf Binax Result</target>
+        <target>Resultados de csf Binax </target>
         <jms:reference-file line="24">views/Report/fieldPopulation.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="abbbc28952e061627f8fd4ad474944cec585a2e7" resname="csf Collected">
         <source>csf Collected</source>
-        <target state="new">csf Collected</target>
+        <target>Coleccionado de csf </target>
         <jms:reference-file line="19">views/Report/fieldPopulation.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="d933c100a3344322c1679817f118083d26a627a7" resname="csf Lat Recorded">
         <source>csf Lat Recorded</source>
-        <target state="new">csf Lat Recorded</target>
+        <target>csf Lat Recorded</target>
         <jms:reference-file line="25">views/Report/fieldPopulation.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="78a8804d05d15573c53379b34f5c7febef6b507b" resname="csf Result">
         <source>csf Result</source>
-        <target state="new">csf Result</target>
+        <target>Resultado de CSF</target>
         <jms:reference-file line="20">views/Report/fieldPopulation.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="3c363836cf4e16666669a25da280a1865c2d2874" resname="d">
@@ -1539,12 +1546,12 @@
       </trans-unit>
       <trans-unit id="24b15c6d3fada6fede239bdff107834b3a0e9990" resname="external-lab-form.comment">
         <source>external-lab-form.comment</source>
-        <target state="new">external-lab-form.comment</target>
+        <target>Comentario</target>
         <jms:reference-file line="28">Form/Rota/BaseLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="bb9ba6fe87ecaac105f0f4af2c061a1748acd534" resname="external-lab-form.date-received">
         <source>external-lab-form.date-received</source>
-        <target state="new">external-lab-form.date-received</target>
+        <target>Fecha en que se recibió la muestra</target>
         <jms:reference-file line="21">Form/Rota/BaseLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="187902eabaf16101d65cb759b954624d7a98de8d" resname="external-lab-form.genotyping-date">
@@ -1579,7 +1586,7 @@
       </trans-unit>
       <trans-unit id="ca99f35dd94f0903815e5b5e5c48596cd446aabf" resname="external-lab-form.pcr-vp6-result">
         <source>external-lab-form.pcr-vp6-result</source>
-        <target state="new">external-lab-form.pcr-vp6-result</target>
+        <target>Resultado del PCR VP6</target>
         <jms:reference-file line="27">Form/Rota/BaseLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="4a0a19218e082a343a1b17e5333409af9d98f0f5" resname="f">
@@ -1603,197 +1610,197 @@
       </trans-unit>
       <trans-unit id="6da3c939c937b6fbd2bf66c6ae49de543af70162" resname="ibd-form.adm-date">
         <source>ibd-form.adm-date</source>
-        <target state="new">ibd-form.adm-date</target>
+        <target>Fecha de admisión </target>
         <jms:reference-file line="112">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="58">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="0c754882743ea4d68accdb243e860ef6c9613576" resname="ibd-form.adm-dx">
         <source>ibd-form.adm-dx</source>
-        <target state="new">ibd-form.adm-dx</target>
+        <target>Diagnostico en la fecha de admisión</target>
         <jms:reference-file line="116">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="59">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="39e83df6a2fb662a0a4eefc0e56eec1be7db70ad" resname="ibd-form.adm-dx-other">
         <source>ibd-form.adm-dx-other</source>
-        <target state="new">ibd-form.adm-dx-other</target>
+        <target>Si el diagnostico en la fecha de admisión es "Otra", por favor especifique </target>
         <jms:reference-file line="120">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="61">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="db64888ca4c5614e098060779e93897f40999089" resname="ibd-form.age">
         <source>ibd-form.age</source>
-        <target state="new">ibd-form.age</target>
+        <target>Edad en meses</target>
         <jms:reference-file line="38">views/RotaVirus/show.html.twig</jms:reference-file>
         <jms:reference-file line="50">views/IBD/show.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="9525a42257a478984411e8019c976f84d268e78b" resname="ibd-form.antibiotics">
         <source>ibd-form.antibiotics</source>
-        <target state="new">ibd-form.antibiotics</target>
+        <target>¿Ha sido el caso tratado con antibióticos la semana pasada?</target>
         <jms:reference-file line="64">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="0e3575c28e3987ae260cba01da2624b9134ba7d1" resname="ibd-form.blood-collect-date">
         <source>ibd-form.blood-collect-date</source>
-        <target state="new">ibd-form.blood-collect-date</target>
+        <target>Fecha de colección de sangre</target>
         <jms:reference-file line="111">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="e112042fbc3747ae7f364fcef4d5cc4341558bba" resname="ibd-form.blood-collected">
         <source>ibd-form.blood-collected</source>
-        <target state="new">ibd-form.blood-collected</target>
+        <target>ibd-form.blood-collected</target>
         <jms:reference-file line="206">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="109">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="da82c27a02bd989de3f189b25fa469f2e4cb2f28" resname="ibd-form.blood-cult-done">
         <source>ibd-form.blood-cult-done</source>
-        <target state="new">ibd-form.blood-cult-done</target>
+        <target>¿Se hizo un cultivo de sangre?</target>
         <jms:reference-file line="233">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="99">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="ef31c56c7c7a3d6b1879e3b389e94787dc00b8c1" resname="ibd-form.blood-cult-other">
         <source>ibd-form.blood-cult-other</source>
-        <target state="new">ibd-form.blood-cult-other</target>
+        <target>Cultivo de sangre Otro</target>
         <jms:reference-file line="236">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="104">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="a898996770506caf8b8cce56b5524c579f3fe285" resname="ibd-form.blood-cult-result">
         <source>ibd-form.blood-cult-result</source>
-        <target state="new">ibd-form.blood-cult-result</target>
+        <target>Resultado del cultivo de sangre</target>
         <jms:reference-file line="102">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="183ff6def325d6c685501a6e7323de0233fbdf30" resname="ibd-form.blood-gram-done">
         <source>ibd-form.blood-gram-done</source>
-        <target state="new">ibd-form.blood-gram-done</target>
+        <target>¿Se hizo una tinción de Gram en el cultivo de sangre?</target>
         <jms:reference-file line="234">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="106">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="b08f0afa7b80ac58861e79763590ceb32034578d" resname="ibd-form.blood-gram-other">
         <source>ibd-form.blood-gram-other</source>
-        <target state="new">ibd-form.blood-gram-other</target>
+        <target>Si otros organismos fueron identificados en la tinción de Gram en el cultivo de sangre, por favor especifique</target>
         <jms:reference-file line="116">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="b680394bb3dd5435545848dda1ed19121637114b" resname="ibd-form.blood-gram-result">
         <source>ibd-form.blood-gram-result</source>
-        <target state="new">ibd-form.blood-gram-result</target>
+        <target>Resultados de la tinción de Gram del cultivo de sangre</target>
         <jms:reference-file line="237">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="108">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="39b803df267a9438df20198b4ebd4acac75bd436" resname="ibd-form.blood-gram-result-organism">
         <source>ibd-form.blood-gram-result-organism</source>
-        <target state="new">ibd-form.blood-gram-result-organism</target>
+        <target>Si organismos fueron detectados en la tinción de Gram del cultivo de sangre, seleccione la opción que mejor describa </target>
         <jms:reference-file line="112">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="22e9fa5667ff6fbb177cb2ffd77f9c9643160c33" resname="ibd-form.blood-id">
         <source>ibd-form.blood-id</source>
-        <target state="new">ibd-form.blood-id</target>
+        <target>Identificación de la muestra de sangre</target>
         <jms:reference-file line="98">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="4a9d08ceeb111a063980989a052ea37fc26f4800" resname="ibd-form.blood-pcr-done">
         <source>ibd-form.blood-pcr-done</source>
-        <target state="new">ibd-form.blood-pcr-done</target>
+        <target>¿Se hizo PCR en el cultivo de sangre?</target>
         <jms:reference-file line="235">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="119">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="270acc56e8c3c3c7955a80eecf46886df992e854" resname="ibd-form.blood-pcr-other">
         <source>ibd-form.blood-pcr-other</source>
-        <target state="new">ibd-form.blood-pcr-other</target>
+        <target>Si otros organismos fueron identificados en PCR en el cultivo de sangre, por favor especifique </target>
         <jms:reference-file line="123">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="4b2ad5f493b7227cc5f880239f2c35e0fad2cf60" resname="ibd-form.blood-pcr-result">
         <source>ibd-form.blood-pcr-result</source>
-        <target state="new">ibd-form.blood-pcr-result</target>
+        <target>Resultados del examen PCR del cultivo de sangre </target>
         <jms:reference-file line="238">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="121">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="e5ba7a020342f19b8f355132f7dd2674f76c52e0" resname="ibd-form.case-id">
         <source>ibd-form.case-id</source>
-        <target state="new">ibd-form.case-id</target>
+        <target>Identificación del caso</target>
         <jms:reference-file line="42">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="57">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="e13e8f9ab84d1d639bf696a7c02cc27e6c2f8379" resname="ibd-form.comment">
         <source>ibd-form.comment</source>
-        <target state="new">ibd-form.comment</target>
+        <target>Comentarios </target>
         <jms:reference-file line="33">Form/IBD/OutcomeType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="62bdde1207c8c4f2f2dbaec50543fd311e98b3e8" resname="ibd-form.csf-appearance">
         <source>ibd-form.csf-appearance</source>
-        <target state="new">ibd-form.csf-appearance</target>
+        <target>Apariencia del CSF</target>
         <jms:reference-file line="202">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="119">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="0f90b164aed657b3c7a0ba7b3800f81a315d6f78" resname="ibd-form.csf-binax-done">
         <source>ibd-form.csf-binax-done</source>
-        <target state="new">ibd-form.csf-binax-done</target>
+        <target>¿Se hizo un examen de Binax en CSF?</target>
         <jms:reference-file line="222">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="67">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="310173f6b5e349ca4849484b0bac14f040c32614" resname="ibd-form.csf-binax-result">
         <source>ibd-form.csf-binax-result</source>
-        <target state="new">ibd-form.csf-binax-result</target>
+        <target>Resultados del examen de Binax para CSF</target>
         <jms:reference-file line="69">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="357445ce7e2e38232bb0f39e5375dfc854d6831a" resname="ibd-form.csf-collect-datetime">
         <source>ibd-form.csf-collect-datetime</source>
-        <target state="new">ibd-form.csf-collect-datetime</target>
+        <target>Fecha y hora de colección de CSF (ejemplo, punción lumbar) del caso</target>
         <jms:reference-file line="198">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="116">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="55a3c923ddac91385faa6c9c10da684e9f63f4a8" resname="ibd-form.csf-collected">
         <source>ibd-form.csf-collected</source>
-        <target state="new">ibd-form.csf-collected</target>
+        <target>¿Se colecciono CSF del caso?</target>
         <jms:reference-file line="194">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="114">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="4e7365004c321f0f7ff7ab3852b3fdd4d3d2a23d" resname="ibd-form.csf-cult-done">
         <source>ibd-form.csf-cult-done</source>
-        <target state="new">ibd-form.csf-cult-done</target>
+        <target>¿Se hizo cultivo en CSF?</target>
         <jms:reference-file line="220">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="45">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="165ff9e8b1253ff274bec7035629a17d6f7d73d2" resname="ibd-form.csf-cult-result">
         <source>ibd-form.csf-cult-result</source>
-        <target state="new">ibd-form.csf-cult-result</target>
+        <target>Resultados del examen de cultivo de  CSF</target>
         <jms:reference-file line="47">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="592e81911f3db51a32610e8f0ba4421adc702f03" resname="ibd-form.csf-culture-contaminant">
         <source>ibd-form.csf-culture-contaminant</source>
-        <target state="new">ibd-form.csf-culture-contaminant</target>
+        <target>ibd-form.csf-cultivo-contaminante</target>
         <jms:reference-file line="52">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="3332d66e09ea8b6dd4eca625c491c87904e6aa2a" resname="ibd-form.csf-culture-other">
         <source>ibd-form.csf-culture-other</source>
-        <target state="new">ibd-form.csf-culture-other</target>
+        <target>Si otros organismos fueron identificados en el cultivo de CSF, por favor especifique </target>
         <jms:reference-file line="225">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="50">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="62b657fd64e41a1266bd381839ac793e57b29053" resname="ibd-form.csf-glucose">
         <source>ibd-form.csf-glucose</source>
-        <target state="new">ibd-form.csf-glucose</target>
+        <target>Medida de la glucosa en CSF</target>
         <jms:reference-file line="218">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="43">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="966e695a098f9080e0d713b2f577f27e6b14c0b8" resname="ibd-form.csf-gram-done">
         <source>ibd-form.csf-gram-done</source>
-        <target state="new">ibd-form.csf-gram-done</target>
+        <target>¿Se hizo la tinción de Gram en el CSF?</target>
         <jms:reference-file line="221">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="54">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="5fc5be4ba1b86fc5db18c5973d5c726f682a3362" resname="ibd-form.csf-gram-other">
         <source>ibd-form.csf-gram-other</source>
-        <target state="new">ibd-form.csf-gram-other</target>
+        <target>Si otros organismos fueron identificados en la tinción de Gram en el CSF, por favor especifique </target>
         <jms:reference-file line="64">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="e92ff3e3870a9074fa68d05de217bf8b5b06f0e8" resname="ibd-form.csf-gram-result">
         <source>ibd-form.csf-gram-result</source>
-        <target state="new">ibd-form.csf-gram-result</target>
+        <target>Resultados de la tinción de Gram del CSF</target>
         <jms:reference-file line="56">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="ba4d2096778797cc7b6d4798c880c485f95da9b9" resname="ibd-form.csf-gram-result-organism">
         <source>ibd-form.csf-gram-result-organism</source>
-        <target state="new">ibd-form.csf-gram-result-organism</target>
+        <target>Si organismos fueron detectados en la tinción de Gram del CSF, Po favor seleccione la opción que mejor describa </target>
         <jms:reference-file line="60">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="57fab7bb44f1759986e31232efcb884b8396d0ed" resname="ibd-form.csf-id">
         <source>ibd-form.csf-id</source>
-        <target state="new">ibd-form.csf-id</target>
+        <target>Identificación de la muestra del CSF</target>
         <jms:reference-file line="41">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="5bd6a0e287156c72e1ca281ce63053256252457c" resname="ibd-form.csf-lab-datetime">
@@ -1803,7 +1810,7 @@
       </trans-unit>
       <trans-unit id="09822becef86dd896f55f4132c0039831a11910e" resname="ibd-form.csf-lat-done">
         <source>ibd-form.csf-lat-done</source>
-        <target state="new">ibd-form.csf-lat-done</target>
+        <target>¿se hizo LAT (látex aglutinación test) en CSF?</target>
         <jms:reference-file line="223">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="71">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
@@ -1944,7 +1951,7 @@
       </trans-unit>
       <trans-unit id="18d965d65f9a42f3d9e87d5d831a1bffe675b70b" resname="ibd-form.hi-serotype">
         <source>ibd-form.hi-serotype</source>
-        <target state="new">ibd-form.hi-serotype</target>
+        <target>La influenza Serotipo, es o no es causada por PCR </target>
         <jms:reference-file line="254">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="90">Form/IBD/SiteLabType.php</jms:reference-file>
       </trans-unit>
@@ -2002,7 +2009,7 @@
       </trans-unit>
       <trans-unit id="467ee04b9fca6cd798a6fb2b5d6c42faae91b21c" resname="ibd-form.men-fontanelle-bulge">
         <source>ibd-form.men-fontanelle-bulge</source>
-        <target state="new">ibd-form.men-fontanelle-bulge</target>
+        <target>El caso, ¿se presentó con fontanela abultada?</target>
         <jms:reference-file line="153">views/IBD/show.html.twig</jms:reference-file>
         <jms:reference-file line="75">Form/IBD/CaseType.php</jms:reference-file>
       </trans-unit>
@@ -2855,7 +2862,7 @@
       </trans-unit>
       <trans-unit id="c405fde50a71cd2c0fa822a260d62d55d5e211cf" resname="site-assigned-case-id">
         <source>site-assigned-case-id</source>
-        <target> ID Sitio centinela</target>
+        <target> ID Sitio Sentinela</target>
         <jms:reference-file line="38">SentinelBundle/Form/CreateType.php</jms:reference-file>
         <jms:reference-file line="39">SentinelBundle/Form/BaseFilter.php</jms:reference-file>
       </trans-unit>
