diff --git a/src/NS/SentinelBundle/Resources/translations/messages.es.xliff b/src/NS/SentinelBundle/Resources/translations/messages.es.xliff
index 700cbfd..5529192 100644
--- a/src/NS/SentinelBundle/Resources/translations/messages.es.xliff
+++ b/src/NS/SentinelBundle/Resources/translations/messages.es.xliff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:jms="urn:jms:translation" version="1.2">
-  <file date="2016-03-28T15:52:44Z" source-language="en" target-language="es" datatype="plaintext" original="not.available">
+  <file date="2016-03-30T16:09:22Z" source-language="en" target-language="es" datatype="plaintext" original="not.available">
     <header>
       <tool tool-id="JMSTranslationBundle" tool-name="JMSTranslationBundle" tool-version="1.1.0-DEV"/>
       <note>The source node in most cases contains the sample message as written by the developer. If it looks like a dot-delimitted string such as "form.label.firstname", then the developer has not provided a default message.</note>
@@ -966,7 +966,7 @@
       </trans-unit>
       <trans-unit id="e8c8fb09a83d86403479132c086fdf365be367fb" resname="Number and Percent Enrolled: Admit Diagnosis">
         <source>Number and Percent Enrolled: Admit Diagnosis</source>
-        <target>Número y porcentaje de inscritos: Administrar diagnosis</target>
+        <target>Número y porcentaje de Registros: Por Diagnosis</target>
         <jms:reference-file line="2">views/Report/percentEnrolled.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="4c100b84b4ee887ba44105eb25d52e008800b694" resname="ORS/ORT and/or IV fluids and/or Other/Multiple">
